# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# shanky <prasad.mvs@gmail.com>, 2012
# shankar <svenkate@redhat.com>, 2006
# shankar <svenkate@redhat.com>, 2008-2012
# shankar <svenkate@redhat.com>, 2013
# Shankar Prasad <svenkte@redhat.com>, 2013
# shankar <svenkate@redhat.com>, 2006,2013
# shanky <prasad.mvs@gmail.com>, 2012
# Vit Mojzis <vmojzis@redhat.com>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-10-31 20:57+0100\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2017-09-01 08:32-0400\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Kannada (http://www.transifex.com/projects/p/fedora/language/"
"kn/)\n"
"Language: kn\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Zanata 3.9.6\n"

#: ../src/allow_anon_write.py:27
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.\n"
"    "
msgstr ""
"\n"
"    SELinux ಪಾಲಿಸಿಯು ಒಂದು http ಸ್ಕ್ರಿಪ್ಟ್‍  ಒಂದು ಸಾರ್ವಜನಿಕ ಕಡತಕೋಶಕ್ಕೆ "
"ಬರೆಯದಂತೆ\n"
"    ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/allow_anon_write.py:32
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.  If httpd is not setup to write to public directories, this\n"
"    could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux ಪಾಲಿಸಿಯು ಒಂದು http ಸ್ಕ್ರಿಪ್ಟ್‍  ಒಂದು ಸಾರ್ವಜನಿಕ ಕಡತಕೋಶಕ್ಕೆ "
"ಬರೆಯದಂತೆ\n"
"    ತಡೆಯುತ್ತಿದೆ.  ಸಾರ್ವಜನಿಕ ಕಡತಕೋಶಗಳಿಗೆ ಬರೆಯಲು httpd ಯನ್ನು ಹೊಂದಿಸದೆ ಹೋದಲ್ಲಿ, "
"ಇದು\n"
"    ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲ್ಪಡುತ್ತದೆ.\n"
"    "

#: ../src/allow_anon_write.py:38
msgid ""
"\n"
"    If httpd scripts should be allowed to write to public directories you "
"need to turn on the $BOOLEAN boolean and change the file context of the "
"public directory to public_content_rw_t.  Read the httpd_selinux\n"
"    man page for further information:\n"
"    \"setsebool -P $BOOLEAN=1; chcon -t public_content_rw_t <path>\"\n"
"    You must also change the default file context labeling files on the "
"system in order to preserve public directory labeling even on a full relabel."
"  \"semanage fcontext -a -t public_content_rw_t <path>\"\n"
"    "
msgstr ""
"\n"
"    httpd ಸ್ಕ್ರಿಪ್ಟುಗಳು ಸಾರ್ವಜನಿಕ ಕೋಶಗಳಿಗೆ ಬರೆಯುವುದನ್ನು ಅನುಮತಿಸಬೇಕಿದ್ದಲ್ಲಿ "
"ನೀವು scripts should be allowed to write to public directories you need to "
"turn on the $BOOLEAN ಬೂಲಿಯನ್ ಅನ್ನು ಚಾಲನೆಗೊಳಿಸಬೇಕು ಮತ್ತು ಸಾರ್ವಜನಿಕ ಕೋಶದ  ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು public_content_rw_t ಗೆ ಬದಲಿಸಬೇಕು.  ಹೆಚ್ಚಿನ ಮಾಹಿತಿಗಾಗಿ "
"httpd_selinux\n"
"    ಮಾಹಿತಿ ಪುಟವನ್ನು ನೋಡಿ:\n"
"    \"setsebool -P $BOOLEAN=1; chcon -t public_content_rw_t <path>\"\n"
"    ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್‌ನ ನಂತರವೂ ಸಾರ್ವಜನಿಕ ಕೋಶದ ಲೇಬಲಿಂಗ್ ಅನ್ನು ಸಂರಕ್ಷಿಸಲು "
"ವ್ಯವಸ್ಥೆಯಲ್ಲಿನ ಪೂರ್ವನಿಯೋಜಿತ ಕಡತ ಸನ್ನಿವೇಶಗಳ ಲೇಬಲಿಂಗ್ ಅನ್ನೂ ಸಹ "
"ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t public_content_rw_t <path>\"\n"
"    "

#: ../src/allow_anon_write.py:44
msgid ""
"If you want to allow $SOURCE_PATH to be able to write to shared public "
"content"
msgstr ""

#: ../src/allow_anon_write.py:45
msgid ""
"you need to change the label on $TARGET_PATH to public_content_rw_t, and "
"potentially turn on the allow_httpd_sys_script_anon_write boolean."
msgstr ""
"$TARGET_PATH ನಲ್ಲಿನ ಲೇಬಲ್ ಅನ್ನು public_content_rw_t ಗೆ ಬದಲಾಯಿಸಬೇಕು, ಹಾಗು "
"allow_httpd_sys_script_anon_write ಬೂಲಿಯನ್ ಅನ್ನು ಆನ್ ಮಾಡಬೇಕು."

#: ../src/allow_execheap.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from changing the access\n"
"    protection of memory on the heap.\n"
"    "
msgstr ""
"\n"
"    $SOURCE_PATH ಅನ್ನು ಒಟ್ಟಿಗೆ ಮೆಮೊರಿಯ ನಿಲುಕಣಾ ಸಂರಕ್ಷಣೆಯನ್ನು ಬದಲಾಯಿಸದಂತೆ\n"
"    SELinux ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/allow_execheap.py:32
msgid ""
"\n"
"    The $SOURCE application attempted to change the access protection of "
"memory on\n"
"    the heap (e.g., allocated using malloc).  This is a potential security\n"
"    problem.  Applications should not be doing this. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not "
"work and\n"
"    you need it to work, you can configure SELinux temporarily to allow\n"
"    this access until the application is fixed. Please file a bug\n"
"    report against this package.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಅನ್ವಯವು ಒಟ್ಟಿಗೆ ಮೆಮೊರಿಯ ನಿಲುಕಣಾ ರಕ್ಷಣೆಯನ್ನು ಬದಲಾಯಿಸಲು "
"ಪ್ರಯತ್ನಿಸಿದೆ\n"
"    (ಉದಾ., malloc ಬಳಸಿಕೊಂಡಿ ನಿಯೋಜಿಸಲಾಗಿದ್ದು). ಇದೊಂದು ದೊಡ್ಡದಾದ ಸುರಕ್ಷತಾ ಲೋಪ.\n"
"    ಅನ್ವಯಗಳು ಹೀಗೆ ಮಾಡಬಾರದು. ಅನ್ವಯಗಳು ಕೆಲವು ಬಾರಿ ತಪ್ಪಾಗಿ ಕೋಡ್ ಮಾಡಲ್ಪಡುವದರಿಂದ\n"
"    ಈ ಬಗೆಯ ಅನುಮತಿಗಾಗಿ ಮನವಿ ಮಾಡುತ್ತವೆ.\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    ಜಾಲ ಪುಟವು ಈ ಅಗತ್ಯವನ್ನು ಹೇಗೆ ತೆಗೆದು ಹಾಕುವುದೆಂದು ವಿವರಿಸುತ್ತದೆ.  ಎಲ್ಲಿಯಾದರೂ "
"$SOURCE ಕೆಲಸ ಮಾಡದೆ ಇದ್ದು ಹಾಗು\n"
"    ನಿಮಗೆ ಅದು ಕೆಲಸ ಮಾಡುವುದುಬೇಕಿದ್ದಲ್ಲಿ, ಅನ್ವಯದ ತೊಂದರೆಯು ನಿವಾರಣೆಯಾಗುವವರೆಗೆ "
"SELinux ಅನ್ನು ತಾತ್ಕಾಲಿಕವಾಗಿ ನೀವು ಈ ಅನುಮತಿಯನ್ನು ಒದಗಿಸುವಂತೆ ಸಂರಚಿಸಬಹುದು.\n"
"    ದಯವಿಟ್ಟು ಈ ಪ್ಯಾಕೇಜಿನ ವಿರುದ್ದ ಒಂದು ದೋಷ\n"
"    ವರದಿಯನ್ನು ಸಲ್ಲಿಸಿ.\n"
"    "

#: ../src/allow_execheap.py:44
msgid ""
"\n"
"    If you want $SOURCE to continue, you must turn on the\n"
"    $BOOLEAN boolean.  Note: This boolean will affect all applications\n"
"    on the system.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಮುಂದುವರೆಯ ಬೇಕಿದ್ದರೆ, ನೀವು\n"
"    $BOOLEAN ಬೂಲಿಯನ್ ಅನ್ನು ಆನ್ ಮಾಡಬೇಕಾಗುತ್ತ್ದೆ.  ಸೂಚನೆ: ಈ ಬೂಲಿಯನ್ ಗಣಕದಲ್ಲಿನ "
"ಎಲ್ಲಾ ಅನ್ವಯಗಳ\n"
"    ಮೇಲೂ ಪರಿಣಾಮ ಬೀರುತ್ತದೆ.\n"
"    "

#: ../src/allow_execheap.py:50
msgid ""
"If you do not think $SOURCE_PATH should need to map heap memory that is both "
"writable and executable."
msgstr ""

#: ../src/allow_execheap.py:51
msgid "you need to report a bug. This is a potentially dangerous access."
msgstr "ನೀವು ಒಂದು ದೋಷವನ್ನು ವರದಿ ಮಾಡಬೇಕಾಗುತ್ತದೆ. ಇದು ಒಂದು ಅಪಾಯಕಾರಿಯಾದ ನಿಲುಕಣೆ."

#: ../src/allow_execheap.py:52 ../src/allow_execstack.py:91
#: ../src/kernel_modules.py:46 ../src/mmap_zero.py:48
#: ../src/selinuxpolicy.py:46 ../src/setenforce.py:29 ../src/setenforce.py:50
#: ../src/sys_module.py:30 ../src/sys_module.py:48
msgid "Contact your security administrator and report this issue."
msgstr "ನಿಮ್ಮ ಸುರಕ್ಷತಾ ವ್ಯವಸ್ಥಾಪಕರನ್ನು ಸಂಪರ್ಕಿಸಿ ಹಾಗು ಈ ವಿಷಯವನ್ನು ವರದಿ ಮಾಡಿ."

#: ../src/allow_execmod.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from loading $TARGET_PATH which "
"requires text relocation.\n"
"    "
msgstr ""
"\n"
"    ಪಠ್ಯ ಸ್ಥಳಾಂತರದ ಅಗತ್ಯವಿರುವ $TARGET_PATH ಅನ್ನು ಲೋಡ್‍ ಆಗದಂತೆ $SOURCE_PATH "
"ಅನ್ನು SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/allow_execmod.py:33
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries do not need this permission. Libraries are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  You can configure\n"
"    SELinux temporarily to allow $TARGET_PATH to use relocation as a\n"
"    workaround, until the library is fixed. Please file a\n"
"bug report.\n"
"    "
msgstr ""

#: ../src/allow_execmod.py:45
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries should not need this permission.   The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">\n"
"    SELinux Memory Protection Tests</a>\n"
"    web page explains this check.  This tool examined the library and it "
"looks\n"
"    like it was built correctly. So setroubleshoot can not determine if this\n"
"    application is compromised or not.  This could be a serious issue. Your\n"
"    system may very well be compromised.\n"
"\n"
"    Contact your security administrator and report this issue.\n"
"\n"
"    "
msgstr ""

#: ../src/allow_execmod.py:62
msgid ""
"\n"
"    If you trust $TARGET_PATH to run correctly, you can change the\n"
"    file context to textrel_shlib_t. \"chcon -t textrel_shlib_t\n"
"    '$TARGET_PATH'\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"textrel_shlib_t '$FIX_TARGET_PATH'\"\n"
"\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:60
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from making the program stack "
"executable.\n"
"    "
msgstr ""
"\n"
"    ಪ್ರೋಗ್ರಾಂ ಸ್ಟಾಕ್ ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸದಂತೆ $SOURCE_PATH ಅನ್ನು SELinux "
"ತಡೆದಿದೆ.\n"
"    "

#: ../src/allow_execstack.py:64
msgid ""
"\n"
"    The $SOURCE application attempted to make its stack\n"
"    executable.  This is a potential security problem.  This should\n"
"    never ever be necessary. Stack memory is not executable on most\n"
"    OSes these days and this will not change. Executable stack memory\n"
"    is one of the biggest security problems. An execstack error might\n"
"    in fact be most likely raised by malicious code. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not\n"
"    work and you need it to work, you can configure SELinux\n"
"    temporarily to allow this access until the application is fixed. Please\n"
"file a bug report.\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:79
msgid ""
"\n"
"    Sometimes a library is accidentally marked with the execstack flag,\n"
"    if you find a library with this flag you can clear it with the\n"
"    execstack -c LIBRARY_PATH.  Then retry your application.  If the\n"
"    app continues to not work, you can turn the flag back on with\n"
"    execstack -s LIBRARY_PATH.\n"
"    "
msgstr ""

#: ../src/allow_execstack.py:89
msgid ""
"If you do not think $SOURCE_PATH should need to map stack memory that is "
"both writable and executable."
msgstr ""

#: ../src/allow_execstack.py:90
msgid "you need to report a bug. \n"
"This is a potentially dangerous access."
msgstr ""
"ನೀವು ಒಂದು ದೋಷವನ್ನು ವರದಿ ಮಾಡಬೇಕಾಗುತ್ತದೆ.  \n"
"ಇದು ಒಂದು ಅಪಾಯಕಾರಿಯಾದ ನಿಲುಕಣೆ."

#: ../src/allow_execstack.py:99
#, python-format
msgid "If you believe that \n"
"%s\n"
"should not require execstack"
msgstr ""

#: ../src/allow_execstack.py:108
#, python-format
msgid ""
"you should clear the execstack flag and see if $SOURCE_PATH works correctly.\n"
"Report this as a bug on %s.\n"
"You can clear the exestack flag by executing:"
msgstr ""
"execstack ಫ್ಲ್ಯಾಗ್ ಅನ್ನು ತೆರವುಗೊಳಿಸಬೇಕು ಹಾಗು $SOURCE_PATH ಸರಿಯಾಗಿ ಕೆಲಸ "
"ಮಾಡುತ್ತದೆಯೆ ಎಂದು ನೋಡಬೇಕು. \n"
"%s ಅನ್ನು ಇದನ್ನು ಒಂದು ದೋಷ ವರದಿಯಾಗಿ ಸಲ್ಲಿಸಿ.\n"
"ಇದನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ exestack ಫ್ಲ್ಯಾಗ್ ಅನ್ನು ತೆರವುಗೊಳಿಸಿ:"

#: ../src/allow_execstack.py:118
#, python-format
msgid "execstack -c %s"
msgstr "execstack -c %s"

#: ../src/allow_ftpd_use_cifs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    "
msgstr ""
"\n"
"     ftp ಡೆಮನ್ ಅನ್ನು CIFS ಕಡತವ್ಯವಸ್ಥೆಯಲ್ಲಿ ಶೇಖರಿಸಲ್ಪಟ್ಟ $ACCESS ಕಡತಗಳನ್ನು "
"ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    CIFS (Comment Internet File System) is a network filesystem similar to\n"
"    SMB (<a href=\"http://www.microsoft.com/mind/1196/cifs.asp\">http://www."
"microsoft.com/mind/1196/cifs.asp</a>)\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As CIFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a CIFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:46
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1.\"\n"
"    "
msgstr ""
"\n"
"    \"$BOOLEAN\"ಬೂಲಿಯನ್ ಅನ್ನುನಿಜಕ್ಕೆ ಬದಲಾಯಿಸುವುದರಿಂದ ಈ ಅನುಮತಿ ನೀಡಲ್ಪಡುತ್ತದೆ:\n"
"    \"setsebool -P $BOOLEAN=1.\"\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:53
msgid ""
" Changing the \"$BOOLEAN\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on CIFS filesystems.  "
msgstr ""
"  \"$BOOLEAN\" ಹಾಗು \"$WRITE_BOOLEAN\"\n"
"     ಬೂಲಿಯನ್‍ಗಳನ್ನು ನಿಜಕ್ಕೆ ಬದಲಾಯಿಸುವುದರಿಂದ ಅದು ಈ ನಿಲುಕನ್ನು ಅನುಮತಿಸುತ್ತದೆ:\n"
"    \"setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1\".\n"
"    ಎಚ್ಚರಿಕೆ: \"$WRITE_BOOLEAN\" ಬೂಲಿಯನ್‍ಗಳನ್ನು ನಿಜಕ್ಕೆ ಹೊಂದಿಸುವುದರಿಂದ\n"
"    ftp ಡೆಮನ್‍ಗೆ CIFS ಕಡತ ವ್ಯವಸ್ಥೆಗಳಲ್ಲಿನ ಕಡತಗಳು ಹಾಗು ಕಡತಕೋಶಗಳ ಜೊತೆಗೆ ಎಲ್ಲಾ\n"
"    ಸಾರ್ವಜನಿಕ ವಿಷಯಗಳನ್ನು (public_content_t ಬಗೆಯ ಕಡತಗಳು ಹಾಗು ಕಡತಕೋಶಗಳು)\n"
"    ಬರೆಯಲು ಅನುಮತಿಯನ್ನು ಒದಗಿಸುತ್ತದೆ.  "

#: ../src/allow_ftpd_use_cifs.py:62
msgid "If you want to allow ftpd to write to cifs file systems"
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:63 ../src/allow_ftpd_use_nfs.py:65
msgid "you must tell SELinux about this"
msgstr "ನೀವು SELinux ಗೆ ಇದರ ಬಗ್ಗೆ ಹೇಳಬೇಕು"

#: ../src/allow_ftpd_use_nfs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    "
msgstr ""
"\n"
"    ftp ಡೆಮನ್ ಅನ್ನು ಒಂದು NFS ಕಡತವ್ಯವಸ್ಥೆಯಲ್ಲಿ ಶೇಖರಿಸಲ್ಪಟ್ಟ $ACCESS ಕಡತಗಳನ್ನು "
"ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/allow_ftpd_use_nfs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    NFS (Network Filesystem) is a network filesystem commonly used on Unix / "
"Linux\n"
"    systems.\n"
"\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As NFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a NFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/allow_ftpd_use_nfs.py:47
msgid ""
"\n"
"    Changing the \"allow_ftpd_use_nfs\" boolean to true will allow this "
"access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1.\"\n"
"    "
msgstr ""
"\n"
"    ಈ ನಿಲುಕಣೆಯನ್ನು ಅನುಮತಿಸಲು \"allow_ftpd_use_nfs\" ಬೂಲಿಯನ್ ಅನ್ನು ನಿಜ ಎಂದು "
"ಬದಲಾಯಿಸಲಾಗುತ್ತಿದೆ:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1.\"\n"
"    "

#: ../src/allow_ftpd_use_nfs.py:54
msgid ""
" Changing the \"allow_ftpd_use_nfs\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on NFS filesystems.  "
msgstr ""
"  \"allow_ftpd_use_nfs\" ಹಾಗು \"$WRITE_BOOLEAN\"\n"
"     ಬೂಲಿಯನ್‍ಗಳನ್ನು ನಿಜಕ್ಕೆ ಬದಲಾಯಿಸುವುದರಿಂದ ಅದು ಈ ನಿಲುಕನ್ನು ಅನುಮತಿಸುತ್ತದೆ:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1\".\n"
"    ಎಚ್ಚರಿಕೆ: \"$WRITE_BOOLEAN\" ಬೂಲಿಯನ್‍ಗಳನ್ನು ನಿಜಕ್ಕೆ ಹೊಂದಿಸುವುದರಿಂದ\n"
"    ftp ಡೆಮನ್‍ಗೆ NFS ಕಡತ ವ್ಯವಸ್ಥೆಗಳಲ್ಲಿನ ಕಡತಗಳು ಹಾಗು ಕಡತಕೋಶಗಳ ಜೊತೆಗೆ ಎಲ್ಲಾ\n"
"    ಸಾರ್ವಜನಿಕ ವಿಷಯಗಳನ್ನು (public_content_t ಬಗೆಯ ಕಡತಗಳು ಹಾಗು ಕಡತಕೋಶಗಳು)\n"
"    ಬರೆಯಲು ಅನುಮತಿಯನ್ನು ಒದಗಿಸುತ್ತದೆ.  "

#: ../src/allow_ftpd_use_nfs.py:64
msgid "If you want to allow ftpd to write to nfs file systems"
msgstr ""

#: ../src/associate.py:32 ../src/openvpn.py:31 ../src/public_content.py:27
#: ../src/qemu_blk_image.py:27 ../src/qemu_file_image.py:27
#: ../src/restorecon.py:51 ../src/restorecon_source.py:33
#: ../src/rsync_data.py:27 ../src/sshd_root.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    ಸಾಧನ $TARGET_PATH ಕ್ಕೆ $SOURCE_PATH \"$ACCESS\" ನ ನಿಲುಕಣೆಯನ್ನು SELinux "
"ತಡೆದಿದೆ. \n"
"    "

#: ../src/associate.py:37
#, python-format
msgid ""
"\n"
"You tried to place a type on a %s that is not a file type.  This is not "
"allowed, you must assigne a file type.  You can list all file types using "
"the seinfo command.\n"
"\n"
"seinfo -afile_type -x\n"
"\n"
"    "
msgstr ""
"\n"
"ನೀವು ಒಂದು ಕಡತದ ಬಗೆಯಾಗಿರದ %s ಎನ್ನುವುದರ ಮೇಲೆ ಒಂದು ಬಗೆಯನ್ನು ಇರಿಸಲು "
"ಪ್ರಯತ್ನಿಸಿದ್ದೀರಿ.  ಇದಕ್ಕೆ ಅನುಮತಿ ಇರುವುದಿಲ್ಲ, ನೀವು ಒಂದು ಕಡತದ ಬಗೆಯನ್ನು "
"ನಿಯೋಜಿಸಬೇಕು. seinfo ಆದೇಶವನ್ನು ಬಳಸಿಕೊಂಡು ನೀವು ಎಲ್ಲಾ ಬಗೆಯ ಕಡತಗಳನ್ನು ನೋಡಬಹುದು.\n"
"\n"
"seinfo -afile_type -x\n"
"\n"
"    "

#: ../src/associate.py:44
#, python-format
msgid ""
"If you want to change the label of $TARGET_PATH to %s, you are not allowed "
"to since it is not a valid file type."
msgstr ""

#: ../src/associate.py:49
msgid "you must pick a valid file label."
msgstr "ನೀವು ಒಂದು ಮಾನ್ಯವಾದ ಕಡತದ ಹೆಸರನ್ನು ಆರಿಸಬೇಕಾಗುತ್ತದೆ."

#: ../src/automount_exec_config.py:27
msgid ""
"\n"
"    SELinux is preventing the $SOURCE_PATH from executing potentially "
"mislabeled files $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    ಸಮರ್ಥವಾಗಿ ತಪ್ಪಾಗಿ ಲೇಬಲ್ ಮಾಡಲ್ಪಟ್ಟ ಕಡತಗಳ $TARGET_PATH ಅನ್ನು "
"$SOURCE_PATHಬಳಸಿಕೊಳ್ಳದಂತೆ SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/automount_exec_config.py:31
msgid ""
"\n"
"    SELinux has denied the $SOURCE_PATH from executing potentially\n"
"    mislabeled files $TARGET_PATH.  Automounter can be setup to execute\n"
"    configuration files. If $TARGET_PATH is an automount executable\n"
"    configuration file it needs to have a file label of bin_t.\n"
"    If automounter is trying to execute something that it is not supposed "
"to, this could indicate an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    ಸಮರ್ಥವಾಗಿ ತಪ್ಪುಲೇಬಲ್ ಮಾಡಲಾದ $TARGET_PATH ಕಡತಗಳನ್ನು $SOURCE_PATH\n"
"    ಕಾರ್ಯಗತಗೊಳಿಸುವುದನ್ನು SELinux ತಡೆಯುತ್ತಿದೆ. ಸಂರಚನಾ ಕಡತಗಳನ್ನು "
"ಕಾರ್ಯಗತಗೊಳಿಸಲು\n"
"    ಆಟೋಕೌಂಟರನ್ನು ಸಜ್ಜುಗೊಳಿಸಬಹುದು. $TARGET_PATH ವು ಸ್ವಯಂಆರೋಹಣಾ(ಆಟೋಮೌಂಟ್)\n"
"    ಕಾರ್ಯಗತಗೊಳಿಸಬಲ್ಲ ಸಂರಚನಾ ಕಡತವಾಗಿದ್ದರೆ ಅದಕ್ಕೆ ಒಂದು bin_t ಕಡತ ಲೇಬಲ್‍ನ "
"ಅಗತ್ಯವಿರುತ್ತದೆ.\n"
"    ಎಲ್ಲಿಯಾದರೂ ಆಟೋಕೌಂಟರ್ ತಾನು ಕಾರ್ಯಗತಗೊಳಿಸಬಾರದುದನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಲು "
"ಪ್ರಯತ್ನಿಸಿದರೆ,\n"
"    ಇದು ಒಂದು ಒಂದು ಒಳನುಸುಳುವಿಕಾ ದಾಳಿ ಎಂದು ಸೂಚಿಸಲ್ಪಡುತ್ತದೆ.\n"
"    "

#: ../src/automount_exec_config.py:39
msgid ""
"\n"
"    If you want to change the file context of $TARGET_PATH so that the "
"automounter can execute it you can execute \"chcon -t bin_t $TARGET_PATH\".  "
"If you want this to survive a relabel, you need to permanently change the "
"file context: execute  \"semanage fcontext -a -t bin_t '$FIX_TARGET_PATH'\".\n"
"    "
msgstr ""
"\n"
"    ಸ್ವಯಂಆರೋಹಕ $TARGET_PATH ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವಂತೆ ಕಡತದ ಸನ್ನಿವೇಶವನ್ನು ನೀವು "
"ಬದಲಾಯಿಸಬೇಕೆಂದರೆ \"chcon -t bin_t $TARGET_PATH'\" ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಬಹುದಾಗಿದೆ."
"ಇದನ್ನು ಒಂದು ಮರುಲೇಬಲ್ ನಂತರವೂ ಉಳಿಯುವಂತೆ ನೀವು ಬಯಸಿದರೆ, ಶಾಶ್ವತವಾಗಿ ಕಡತ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ: ಇದಕ್ಕಾಗಿ  \"semanage fcontext -a -t bin_t "
"'$FIX_TARGET_PATH'\"ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಿ.\n"
"    "

#: ../src/bind_ports.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from binding to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    $SOURCE_PATH ಅನ್ನು ಸಂಪರ್ಕಸ್ಥಾನ $PORT_NUMBER ಕ್ಕೆ ಬೈಂಡ್ ಮಾಡದಂತೆ SELinux "
"ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/bind_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE from binding to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to listen on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can bind to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to bind to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/bind_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to bind to port $PORT_NUMBER, you can "
"execute \n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    \n"
"where PORT_TYPE is one of the following: %s.\n"
"\n"
"    \n"
"\n"
"If this system is running as an NIS Client, turning on the allow_ypbind "
"boolean may fix the problem.  setsebool -P allow_ypbind=1.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಗೆ ಸಂಪರ್ಕಸ್ಥಾನ $PORT_NUMBER ಬೈಂಡ್ ಆಗಿರಬೇಕೆಂದು ನೀವು ಬಯಸಿದಲ್ಲಿ, \n"
"   # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER ಅನ್ನು "
"ಕಾರ್ಯಗತಗೊಳಿಸಬಹುದು\n"
"    ಇಲ್ಲಿ PORT_TYPE ಎನ್ನುವುದು ಈ ಕೆಳಗಿನವುಗಳಲ್ಲಿ ಒಂದಾಗಿರುತ್ತದೆ: %s.\n"
"\n"
"    ಈ ಗಣಕವು ಒಂದು NIS ಕ್ಲೈಂಟ್ ಆಗಿ ಚಲಾಯಿತಗೊಳ್ಳುತ್ತಿದ್ದಲ್ಲಿ, allow_ypbind "
"ಬೂಲಿಯನ್ ಅನ್ನು ಆನ್ ಮಾಡುವುದರಿಂದ, ಈ ತೊಂದರೆಯನ್ನು ಪರಿಹರಿಸಬಹುದಾಗಿದೆ.  setsebool -P "
"allow_ypbind=1.\n"
"    "

#: ../src/bind_ports.py:47
msgid "If you want to allow $SOURCE_PATH to bind to network port $PORT_NUMBER"
msgstr ""

#: ../src/bind_ports.py:53 ../src/connect_ports.py:51
#, python-format
msgid ""
"# semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    where PORT_TYPE is one of the following: %s."
msgstr ""
"#semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    ಇಲ್ಲಿ PORT_TYPE ಎನ್ನುವುದ ಈ ಕೆಳಗಿನವುಗಳಲ್ಲಿ ಒಂದಾಗಿರುತ್ತದೆ: %s.    "

#: ../src/bind_ports.py:56 ../src/connect_ports.py:54
#, python-format
msgid "# semanage port -a -t %s -p %s $PORT_NUMBER"
msgstr "# semanage port -a -t %s -p %s $PORT_NUMBER"

#: ../src/catchall_boolean.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access on $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    ಸಾಧನ $TARGET_PATH ಗೆ $SOURCE_PATH \"$ACCESS\" ನ ನಿಲುಕಣೆಯನ್ನು SELinux "
"ತಡೆದಿದೆ. \n"
"    "

#: ../src/catchall_boolean.py:36
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. The current boolean\n"
"    settings do not allow this access.  If you have not setup $SOURCE to\n"
"    require this access this may signal an intrusion attempt. If you do "
"intend\n"
"    this access you need to change the booleans on this system to allow\n"
"    the access.\n"
"    "
msgstr ""

#: ../src/catchall_boolean.py:45
msgid ""
"\n"
"    Confined processes can be configured to run requiring different access, "
"SELinux provides booleans to allow you to turn on/off\n"
"    access as needed.\n"
"\n"
"    "
msgstr ""

#: ../src/catchall_boolean.py:62
#, python-format
msgid "If you want to %s"
msgstr ""

#: ../src/catchall_boolean.py:65
#, python-format
msgid "setsebool -P %s %s"
msgstr "setsebool -P %s %s"

#: ../src/catchall_boolean.py:68
#, python-format
msgid "You must tell SELinux about this by enabling the '%s' boolean.\n"
msgstr ""
"'%s' ಬೂಲಿಯನ್ ಅನ್ನು ಶಕ್ತಗೊಳಿಸುವ ಮೂಲಕ ನೀವು SELinux ಗೆ ಇದರ ಬಗ್ಗೆ ಹೇಳಬೇಕು.\n"

#: ../src/catchall_boolean.py:71
#, python-format
msgid "You can read '%s' man page for more details."
msgstr "ಹೆಚ್ಚಿನ ವಿವರಗಳಿಗಾಗಿ ನೀವು '%s' ಮಾಹಿತಿ ಪುಟವನ್ನು ನೋಡಬಹುದು."

#: ../src/catchall_labels.py:28
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE access to potentially\n"
"    mislabeled files $TARGET_PATH.  This means that SELinux will not\n"
"    allow httpd to use these files. If httpd should be allowed this access "
"to these files you should change the file context to one of the following "
"types, %s.\n"
"    Many third party apps install html files\n"
"    in directories that SELinux policy cannot predict.  These directories\n"
"    have to be labeled with a file context which httpd can access.\n"
"    "
msgstr ""
"\n"
"    ಸಮರ್ಥವಾಗಿ ತಪ್ಪಾಗಿ ಲೇಬಲ್ ಮಾಡಲ್ಪಟ್ಟ ಕಡತಗಳು ($TARGET_PATH) ಅನ್ನು $SOURCE\n"
"    ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ SELinux ತಡೆದಿದೆ. ಇದರರ್ಥ httpd ಯು ಈ ಕಡತಗಳನ್ನು "
"ಬಳಸುವುದನ್ನು\n"
"    SELinux ಅನುಮತಿಸುವುದಿಲ್ಲ ಎಂದಾಗುತ್ತದೆ. ಎಲ್ಲಿಯಾದರೂ ಈ ಕಡತಗಳಿಗೆ httpd ಯು ಈ "
"ನಿಲುಕಣೆಯನ್ನು ಒದಗಿಸಲು ಬಯಸಿದಲ್ಲಿ ಕಡತದ ಸನ್ನಿವೇಶವನ್ನು ಈ ಕೆಳಗಿನವುಗಳಲ್ಲಿ ಒಂದಕ್ಕೆ "
"ಬದಲಾಯಿಸಬೇಕು, %s.\n"
" ಹಲವಾರು ಥರ್ಡ್ ಪಾರ್ಟಿ ಅನ್ವಯಗಳು  SELinux ಊಹಿಸಲು ಸಾಧ್ಯವಾಗದ ಕಡತಕೋಶಗಳಲ್ಲಿ html "
"ಕಡತಗಳನ್ನು ಅನುಸ್ಥಾಪಿಸುತ್ತವೆ.\n"
"    ಈ ಕಡತಕೋಶಗಳನ್ನು httpd ಯು ನಿಲುಕಿಸಿಕೊಳ್ಳುವಂತಹ ಒಂದು ಕಡತ\n"
"    ಸನ್ನಿವೇಶಗಳೊಂದಿಗೆ ಲೇಬಲ್ ಮಾಡಬೇಕಾಗುತ್ತದೆ\n"
"    "

#: ../src/catchall_labels.py:37
msgid "You need to change the label on $FIX_TARGET_PATH"
msgstr "$FIX_TARGET_PATH ನಲ್ಲಿ ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸುವ ಅಗತ್ಯವಿದೆ"

#: ../src/catchall_labels.py:40
#, python-format
msgid ""
"# semanage fcontext -a -t FILE_TYPE '$FIX_TARGET_PATH'\n"
"where FILE_TYPE is one of the following: %s.\n"
"Then execute:\n"
"restorecon -v '$FIX_TARGET_PATH'\n"
msgstr ""

#: ../src/catchall.py:29 ../src/chrome.py:29 ../src/mozplugger.py:29
#: ../src/mozplugger_remove.py:29
msgid "SELinux is preventing $SOURCE_PATH \"$ACCESS\" access."
msgstr "$SOURCE_PATH \"$ACCESS\" ನ ನಿಲುಕಣೆಯನ್ನು SELinux ತಡೆಯುತ್ತಿದೆ."

#: ../src/catchall.py:31
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"\n"
"    "
msgstr ""
"\n"
"\n"
"    $SOURCE ನಿಂದ ಸಲ್ಲಿಸಲಾದ ಮನವಿಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ. \n"
"    ಈ ಅನುಮತಿಯು $SOURCE ಗೆ ಅಗತ್ಯವಿದೆ ಎಂದು ನಿರೀಕ್ಷಿಸಿರಲಿಲ್ಲ ಹಾಗು ಇದನ್ನು\n"
"    ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲ್ಪಡುತ್ತದೆ. ಅನ್ವಯದ ನಿಶ್ಚಿತ ಆವೃತ್ತಿ\n"
"    ಅಥವ ಸಂರಚನೆಗಾಗಿ ಈ ಹೆಚ್ಚುವರಿ ಅನುಮತಿಯ ಅಗತ್ಯವಿರುವ ಸಾಧ್ಯತೆ ಇದೆ.\n"
"\n"
"   "

#: ../src/catchall.py:41
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"\n"
"    Please file a bug report.\n"
"    "
msgstr ""
"\n"
"   ಈ ನಿಲುಕಣೆಯನ್ನು ಅನುಮತಿಸಲು ನೀವು ಒಂದು ಸ್ಥಳೀಯ ಪಾಲಿಸಿಯನ್ನು ಉತ್ಪಾದಿಸಬಹುದು\n"
"    - <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a> ಅನ್ನು ನೋಡಿ\n"
"    ದಯವಿಟ್ಟು ಒಂದು ದೋಷ ವರದಿಯನ್ನು ಸಲ್ಲಿಸಿ.\n"
"    "

#: ../src/catchall.py:50
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"processes labeled $TARGET_TYPE by default."
msgstr ""

#: ../src/catchall.py:52
msgid ""
"If you believe that $SOURCE_BASE_PATH should have the $ACCESS capability by "
"default."
msgstr ""

#: ../src/catchall.py:53
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"the $TARGET_BASE_PATH $TARGET_CLASS by default."
msgstr ""

#: ../src/catchall.py:55
msgid ""
"You should report this as a bug.\n"
"You can generate a local policy module to allow this access."
msgstr ""
"ನೀವು ಇದನ್ನು ಒಂದು ದೋಷ ಎಂದು ವರದಿ ಮಾಡಬೇಕು.\n"
"ಈ ನಿಲುಕಣೆಯನ್ನು ಅನುಮತಿಸಲು ನೀವು ಒಂದು ಸ್ಥಳೀಯ ಪಾಲಿಸಿಯನ್ನು ಉತ್ಪಾದಿಸಬಹುದು."

#: ../src/catchall.py:56
msgid ""
"Allow this access for now by executing:\n"
"# ausearch -c '$SOURCE' --raw | audit2allow -M my-$MODULE_NAME\n"
"# semodule -i my-$MODULE_NAME.pp"
msgstr ""

#: ../src/connect_ports.py:27 ../src/sandbox_connect.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from connecting to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    $SOURCE_PATH ಅನ್ನು ಸಂಪರ್ಕಸ್ಥಾನ $PORT_NUMBER ಕ್ಕೆ ಬರೆಯದಂತೆ SELinux "
"ತಡೆದಿದೆ\n"
"    "

#: ../src/connect_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can connect to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/connect_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"\n"
"    where PORT_TYPE is one of the following: %s.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಅನ್ನು $PORT_NUMBER ಗೆ ಸಂಪರ್ಕಿತಗೊಳ್ಳಲು ನೀವು ಬಯಸಿದಲ್ಲಿ, ಇದನ್ನು "
"ಚಲಾಯಿಸಿ \n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER,\n"
"    <br>ಇಲ್ಲಿ PORT_TYPE ಎನ್ನುವುದ ಈ ಕೆಳಗಿನವುಗಳಲ್ಲಿ ಒಂದಾಗಿರುತ್ತದೆ: %s.\n"
"    "

#: ../src/connect_ports.py:45 ../src/sandbox_connect.py:47
msgid ""
"If you want to allow $SOURCE_PATH to connect to network port $PORT_NUMBER"
msgstr ""

#: ../src/chrome.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  spice-xpi run "
"applications within mozilla-plugins that require access to the desktop, that "
"the mozilla_plugin lockdown will not allow, so either you need to turn off "
"the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""

#: ../src/chrome.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi', or turn off enforcement of SELinux over the Chrome "
"plugins. setsebool -P unconfined_chrome_sandbox_transition 0\n"
"    "
msgstr ""

#: ../src/chrome.py:38 ../src/mozplugger.py:38
#, python-format
msgid "If you want to use the %s package"
msgstr ""

#: ../src/chrome.py:43
msgid "you must turn off SELinux controls on the Chrome plugins."
msgstr ""
"ಕ್ರೋಮ್ ಪ್ಲಗ್‌ಇನ್‌ಗಳಲ್ಲಿ ನೀವು SELinux ನಿಯಂತ್ರಕಗಳನ್ನು ಆಫ್ ಮಾಡಬೇಕಾಗುತ್ತದೆ."

#: ../src/cvs_data.py:27
msgid ""
"\n"
"    SELinux is preventing cvs ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH\n"
"    "
msgstr ""
"\n"
"    ಸಾಧನ $TARGET_PATH ಕ್ಕೆ ($SOURCE_PATH) \"$ACCESS\" ನ ನಿಲುಕಣೆಯನ್ನು SELinux "
"ತಡೆದಿದೆ. \n"
"    "

#: ../src/cvs_data.py:31
msgid ""
"\n"
"    SELinux denied cvs access to $TARGET_PATH.\n"
"    If this is a CVS repository it needs to have a file context label of\n"
"    cvs_data_t. If you did not intend to use $TARGET_PATH as a CVS "
"repository\n"
"    it could indicate either a bug or it could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    $TARGET_PATH ಗೆ SELinux cvs ಅನುಮತಿಯನ್ನು ನಿರಾಕರಿಸಿದೆ.\n"
"    ಎಲ್ಲಿಯಾದರೂ ಇದು ಒಂದು CVS ರೆಪೊಸಿಟರಿಯಾಗಿರ ಬೇಕಿದ್ದಲ್ಲಿ ಇದರ ಕಡತ ಸನ್ನಿವೇಶದ "
"ಲೇಬಲ್ \n"
"    cvs_data_t ಆಗಿರಬೇಕು. ನೀವು $TARGET_PATH ಅನ್ನು ಒಂದು CVS ರೆಪೊಸಿಟರಿಯಾಗಿ \n"
"    ಬಳಸುವ ಬಯಸದೆ ಹೋಗಿದ್ದಲ್ಲಿ, ಇದು ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು "
"ಸೂಚಿಸಲಾಗುತ್ತದೆ.\n"
"    "

#: ../src/cvs_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t cvs_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"cvs_data_t '$FIX_TARGET_PATH'\"\n"
"    \n"
"    "
msgstr ""
"\n"
"    chcon -R -t cvs_data_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"cvs_data_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/cvs_data.py:44
msgid "If $TARGET_BASE_PATH should be shared via the cvs daemon"
msgstr ""

#: ../src/cvs_data.py:45 ../src/qemu_file_image.py:49
msgid "You need to change the label on $TARGET_BASE_PATH'"
msgstr "$TARGET_BASE_PATH' ನಲ್ಲಿ ನೀವು ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ"

#: ../src/dac_override.py:28
msgid "\n"
"    SELinux is preventing $SOURCE_PATH the \"$ACCESS\" capability.\n"
"    "
msgstr ""

#: ../src/dac_override.py:32
msgid ""
"\n"
"\tdac_override and dac_read_search capabilities usually indicates that the "
"root process does not have access to a file based on the permission flags.  "
"This usually mean you have some file with the wrong ownership/permissions on "
"it.\n"
"    "
msgstr ""
"\n"
"\tdac_override ಹಾಗು dac_read_search ಸಾಮರ್ಥ್ಯಗಳು ಅನುಮತಿ ಫ್ಲ್ಯಾಗ್‌ಗಳ ಆಧರಿತವಾದ "
"ಕಡತವನ್ನು ನಿಲುಕಿಸಿಕೊಳ್ಳುವ ಅನುಮತಿಗಳನ್ನು ಹೊಂದಿರುವುದಿಲ್ಲ ಎನ್ನುವುದನ್ನು "
"ಸೂಚಿಸುತ್ತದೆ.  ಸಾಮಾನ್ಯವಾಗಿ ಇದರ ಅರ್ಥ ನೀವು ತಪ್ಪು ಮಾಲಿಕತ್ವ/ಅನುಮತಿಗಳನ್ನು ಹೊಂದಿರುವ "
"ಕೆಲವು ಕಡತಗಳು ನಿಮ್ಮಲ್ಲಿ ಇವೆ ಎನ್ನುವುದಾಗಿರುತ್ತದೆ.\n"
"    "

#: ../src/dac_override.py:38
msgid ""
"If you want to help identify if domain needs this access or you have a file "
"with the wrong permissions on your system"
msgstr ""

#: ../src/dac_override.py:39
msgid ""
"turn on full auditing to get path information about the offending file and "
"generate the error again."
msgstr ""
"ತಪ್ಪೆಸಗುತ್ತಿರುವ ಕಡತವು ಇರುವ ಸ್ಥಳದ ಬಗೆಗೆ ಮಾಹಿತಿಯನ್ನು ಪಡೆಯಲು ಹಾಗು ದೋಷವನ್ನು "
"ಇನ್ನೊಮ್ಮೆ ಉತ್ಪಾದಿಸಲು ಸಂಪೂರ್ಣ ಆಡಿಟಿಂಗ್ ಅನ್ನು ಆನ್ ಮಾಡಿ."

#: ../src/dac_override.py:40
msgid ""
"\n"
"Turn on full auditing\n"
"# auditctl -w /etc/shadow -p w\n"
"Try to recreate AVC. Then execute\n"
"# ausearch -m avc -ts recent\n"
"If you see PATH record check ownership/permissions on file, and fix it,\n"
"otherwise report as a bugzilla."
msgstr ""

#: ../src/device.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to device "
"$TARGET_PATH.\n"
"    "
msgstr ""

#: ../src/device.py:33
msgid ""
"\n"
"\n"
"    SELinux has denied $SOURCE \"$ACCESS\" access to device $TARGET_PATH.\n"
"    $TARGET_PATH is mislabeled, this device has the default label of the /"
"dev directory, which should not\n"
"    happen.  All Character and/or Block Devices should have a label.\n"
"\n"
"    You can attempt to change the label of the file using\n"
"\n"
"    restorecon -v '$TARGET_PATH'.\n"
"\n"
"    If this device remains labeled device_t, then this is a bug in SELinux "
"policy.\n"
"\n"
"    Please file a bg report.\n"
"\n"
"    If you look at the other similar devices labels, ls -lZ /dev/SIMILAR, "
"and find a type that would work for $TARGET_PATH,\n"
"    you can use chcon -t SIMILAR_TYPE '$TARGET_PATH', If this fixes the "
"problem, you can make this permanent by executing\n"
"    semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"\n"
"    If the restorecon changes the context, this indicates that the "
"application that created the device, created it without\n"
"    using SELinux APIs.  If you can figure out which application created the "
"device, please file a bug report against this application.\n"
"\n"
"    "
msgstr ""

#: ../src/device.py:56
msgid ""
"\n"
"    Attempt restorecon -v '$TARGET_PATH' or chcon -t SIMILAR_TYPE "
"'$TARGET_PATH'\n"
"    "
msgstr ""
"\n"
"    restorecon -v '$TARGET_PATH' ಅಥವ chcon -t SIMILAR_TYPE '$TARGET_PATH' ಗೆ "
"ಪ್ರಯತ್ನಿಸಿ\n"
"    "

#: ../src/device.py:60
msgid ""
"You need to change the label on $TARGET_PATH to a type of a similar device."
msgstr ""
"$TARGET_PATH ನಲ್ಲಿ ಇದೇ ಬಗೆಯ ಒಂದು ಸಾಧನಕ್ಕಾಗಿ ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸುವ "
"ಅಗತ್ಯವಿರುತ್ತದೆ."

#: ../src/device.py:61
msgid ""
"# semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"

#: ../src/disable_ipv6.py:30
msgid "\n"
"Disable IPV6 properly.\n"
"    "
msgstr "\n"
"IPV6 ಅನ್ನು ಸರಿಯಾಗಿ ಅಶಕ್ತಗೊಳಿಸಿ.\n"
"    "

#: ../src/disable_ipv6.py:40
msgid "If you want to disable IPV6 on this machine"
msgstr ""

#: ../src/disable_ipv6.py:41
msgid ""
"you need to set /proc/sys/net/ipv6/conf/all/disable_ipv6 to 1 and do not "
"blacklist the module'"
msgstr ""
"ನೀವು /proc/sys/net/ipv6/conf/all/disable_ipv6 ಅನ್ನು 1 ಕ್ಕೆ ಹೊಂದಿಸಬೇಕು ಮತ್ತು "
"ಮಾಡ್ಯೂಲ್ ಅನ್ನು ಕಪ್ಪುಪಟ್ಟಿಗೆ ಸೇರಿಸಬಾರದು'"

#: ../src/disable_ipv6.py:42
msgid "Add \n"
"net.ipv6.conf.all.disable_ipv6 = 1\n"
"to /etc/sysctl.conf\n"
msgstr "/etc/sysctl.conf ಗೆ net.ipv6.conf.all.disable_ipv6 = 1\n"
"ಅನ್ನು ಸೇರಿಸಿ\n"

#: ../src/file.py:27
msgid ""
"\n"
"    SELinux is preventing access to files with the label, file_t.\n"
"    "
msgstr ""
"\n"
"    file_t ಲೇಬಲ್ ಅನ್ನು ಹೊಂದಿರುವ ಕಡತಗಳನ್ನು ನಿಲುಕಿಸಿಕೊಳ್ಳುವುದನ್ನು SELinux "
"ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/file.py:31
msgid ""
"\n"
"    SELinux permission checks on files labeled file_t are being\n"
"    denied.  file_t is the context the SELinux kernel gives to files\n"
"    that do not have a label. This indicates a serious labeling\n"
"    problem. No files on an SELinux box should ever be labeled file_t.\n"
"    If you have just added a disk drive to the system you can\n"
"    relabel it using the restorecon command.  For example if you saved the\n"
"home directory from a previous installation that did not use SELinux, "
"'restorecon -R -v /home' will fix the labels.  Otherwise you should\n"
"    relabel the entire file system.\n"
"    "
msgstr ""
"\n"
"    file_t ಎಂದು ಲೇಬಲ್ ಮಾಡಲಾದ ಕಡತಗಳಿಗೆ SELinux ಅನುಮತಿ ಪರಿಶೀಲನೆಯು "
"ನಿರಾಕರಿಸಲ್ಪಟ್ಟಿದೆ\n"
"    file_t ಎನ್ನುವುದು  ಒಂದು ಲೇಬಲ್ ಇಲ್ಲದ ಕಡತಗಳಿಗೆ SELinux ಕರ್ನಲ್ "
"ಒದಗಿಸಲ್ಪಡುವುದಾಗಿದೆ\n"
"    ಇದು ಒಂದು ದೊಡ್ಡದಾದ ಲೇಬಲಿಂಗ್ ತೊಂದರೆಯಾಗಿದೆ\n"
"    ಒಂದು SELinux ಬಾಕ್ಸ್‍ನಲ್ಲಿನ ಯಾವುದೆ ಕಡತವು file_t ನಿಂದ ಲೇಬಲ್ ಆಗಬಾರದು.\n"
"    ನೀವು ಈಗತಾನೆ ನಿಮ್ಮ ಗಣಕಕ್ಕೆ ಒಂದು ಹೊಸ ಡಿಸ್ಕ್‍ ಅನ್ನು ಸೇರಿಸಿದ್ದಲ್ಲಿ, ನೀವದನ್ನು "
"\n"
"    restorecon ಆದೇಶವನ್ನು ಉಪಯೋಗಿಸಿ ಮರು ಲೇಬಲ್ ಮಾಡಬಹುದಾಗಿದೆ. ಉದಾಹರಣೆಗೆ\n"
"ನೀವು ನೆಲೆ ಕೋಶವನ್ನು SELinux ಬಳಸದೆ ಇರುವ ಹಿಂದಿನ ಅನುಸ್ಥಾಪನೆಯಿಂದ "
"ಉಳಿಸಿಕೊಂಡಿದ್ದಲ್ಲಿ,\n"
"'restorecon -R -v /home' ಎನ್ನುವುದು ಲೇಬಲ್‌ಗಳನ್ನು ಸರಿಪಡಿಸುತ್ತದೆ. ಇಲ್ಲದೆ "
"ಹೋದಲ್ಲಿ \n"
"    ನೀವು ಸಂಪೂರ್ಣ ಕಡತವ್ಯವಸ್ಥೆಯನ್ನು ಮರು ಲೇಬಲ್ ಮಾಡಬೇಕಾಗುತ್ತದೆ.\n"
"    "

#: ../src/file.py:42
msgid ""
"\n"
"    You can execute the following command as root to relabel your\n"
"    computer system: \"touch /.autorelabel; reboot\"\n"
"    "
msgstr ""
"\n"
"    ನಿಮ್ಮ ಗಣಕ ವ್ಯವಸ್ಥೆಯನ್ನು ಮರುಲೇಬಲ್ ಮಾಡಲು ನೀವು ಈ ಕೆಳಗಿನ ಆದೇಶವನ್ನು "
"ಕಾರ್ಯಗತಗೊಳಿಸಬಹುದು:\n"
"    \"touch /.autorelabel; reboot\"\n"
"    "

#: ../src/file.py:49
msgid "This is caused by a newly created file system."
msgstr ""

#: ../src/file.py:51
msgid "If you think this is caused by a badly mislabeled machine."
msgstr ""

#: ../src/file.py:55
msgid "you need to add labels to it."
msgstr "ನೀವು ಅದಕ್ಕೆ ಲೇಬಲ್‌ಗಳನ್ನು ಸೇರಿಸುವ ಅಗತ್ಯವಿರುತ್ತದೆ."

#: ../src/file.py:57
msgid "you need to fully relabel."
msgstr "ನೀವು ಸಂಪೂರ್ಣವಾಗಿ ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ."

#: ../src/filesystem_associate.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from creating a file with a context "
"of $SOURCE_TYPE on a filesystem.\n"
"    "
msgstr ""
"\n"
"    ಒಂದು ಕಡತವ್ಯವಸ್ಥೆಯಲ್ಲಿ $SOURCE_TYPE ನ ಕಡತ ಸನ್ನಿವೇಶದೊಂದಿಗೆ $SOURCE_PATH "
"ಒಂದು ಕಡತವನ್ನು ನಿರ್ಮಿಸುವುದನ್ನು SELinux ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/filesystem_associate.py:31
msgid ""
"\n"
"    SELinux is preventing $SOURCE from creating a file with a context of "
"$SOURCE_TYPE on a filesystem.\n"
"    Usually this happens when you ask the cp command to maintain the context "
"of a file when\n"
"    copying between file systems, \"cp -a\" for example.  Not all file "
"contexts should be maintained\n"
"    between the file systems.  For example, a read-only file type like "
"iso9660_t should not be placed\n"
"    on a r/w system.  \"cp -p\" might be a better solution, as this will "
"adopt the default file context\n"
"    for the destination.\n"
"    "
msgstr ""

#: ../src/filesystem_associate.py:40
msgid ""
"\n"
"    Use a command like \"cp -p\" to preserve all permissions except SELinux "
"context.\n"
"    "
msgstr ""
"\n"
"    SELinux ಸನ್ನಿವೇಶವನ್ನು ಹೊರತುಪಡಿಸಿ ಎಲ್ಲಾ ಅನುಮತಿಗಳನ್ನು ಉಳಿಸಲು \"cp -p\" "
"ಅನ್ನು ಬಳಸಿ.\n"
"    "

#: ../src/filesystem_associate.py:43
msgid ""
"If you believe $SOURCE_BASE_PATH should be allowed to create "
"$TARGET_BASE_PATH files"
msgstr ""

#: ../src/filesystem_associate.py:44
msgid ""
"you need to use a different command. You are not allowed to preserve the "
"SELinux context on the target file system."
msgstr ""
"ನೀವು ಬೇರೊಂದು ಆದೇಶವನ್ನು ಬಳಸಬೇಕಾಗುತ್ತದೆ. ನಿರ್ದೇಶಿತ ವ್ಯವಸ್ಥೆಯಲ್ಲಿ SELinux "
"ಸನ್ನಿವೇಶವನ್ನು ಕಾದಿರಿಸಲು ನಿಮಗೆ ಅನುಮತಿ ಇಲ್ಲ."

#: ../src/filesystem_associate.py:46
msgid ""
"use a command like \"cp -p\" to preserve all permissions except SELinux "
"context."
msgstr ""
"SELinux ಸನ್ನಿವೇಶವನ್ನು ಹೊರತುಪಡಿಸಿ ಎಲ್ಲಾ ಅನುಮತಿಗಳನ್ನು ಉಳಿಸಲು \"cp -p\" ಅನ್ನು "
"ಬಳಸಿ.    "

#: ../src/httpd_can_sendmail.py:28
msgid "\n"
"    SELinux is preventing the http daemon from sending mail.\n"
"    "
msgstr "\n"
"    http ಡೆಮನ್ ಒಂದು ಮೈಲನ್ನು ಕಳುಹಿಸದಂತೆ SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/httpd_can_sendmail.py:32
msgid ""
"\n"
"    SELinux has denied the http daemon from sending mail. An\n"
"    httpd script is trying to connect to a mail port or execute the\n"
"    sendmail command. If you did not setup httpd to sendmail, this could\n"
"    signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/httpd_can_sendmail.py:39
msgid ""
"\n"
"    If you want httpd to send mail you need to turn on the\n"
"    $BOOLEAN boolean: \"setsebool -P\n"
"    $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    httpd ಯು ಮೈಲ್ ಅನ್ನು ಕಳುಹಿಸುವುದನ್ನು ಸಕ್ರಿಯಗೊಳಿಸಲು ನೀವು $BOOLEAN\n"
"    ಬೂಲಿಯನ್ ಅನ್ನು ಆನ್ ಮಾಡಬೇಕಾಗುತ್ತದೆ: \"setsebool -P\n"
"    $BOOLEAN=1\"\n"
"    "

#: ../src/httpd_can_sendmail.py:47
msgid "If you want to allow httpd to send mail"
msgstr ""

#: ../src/httpd_can_sendmail.py:48
msgid "you must setup SELinux to allow this"
msgstr "ಇದಕ್ಕೆ ಅನುಮತಿಸಲು ನೀವು SELinux ಅನ್ನು ಸಿದ್ಧಗೊಳಿಸಬೇಕು"

#: ../src/httpd_unified.py:29
msgid "\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"    "
msgstr ""
"\n"
"    httpd ಯು $ACCESS ಅನ್ನು http ಕಡತಗಳನ್ನು  ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ SELinux "
"ನಿರ್ಬಂಧಿಸಿದೆ.\n"
"    "

#: ../src/httpd_unified.py:33
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"\n"
"    Ordinarily httpd is allowed full access to all files labeled with http "
"file\n"
"    context.  This machine has a tightened security policy with the $BOOLEAN\n"
"    turned off,  This requires explicit labeling of all files.  If a file is\n"
"    a cgi script it needs to be labeled with httpd_TYPE_script_exec_t in "
"order\n"
"    to be executed.  If it is read only content, it needs to be labeled\n"
"    httpd_TYPE_content_t, it is writable content. it needs to be labeled\n"
"    httpd_TYPE_script_rw_t or httpd_TYPE_script_ra_t. You can use the\n"
"    chcon command to change these context.  Please refer to the man page\n"
"    \"man httpd_selinux\" or\n"
"    <a href=\"http://fedora.redhat.com/docs/selinux-apache-fc3\">FAQ</a>\n"
"    \"TYPE\" refers to one of \"sys\", \"user\" or \"staff\" or potentially "
"other\n"
"    script types.\n"
"    "
msgstr ""

#: ../src/httpd_unified.py:50
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    \"$BOOLEAN\"ಬೂಲಿಯನ್ ಅನ್ನುನಿಜಕ್ಕೆ ಬದಲಾಯಿಸುವುದರಿಂದ ಈ ಅನುಮತಿ ನೀಡಲ್ಪಡುತ್ತದೆ:\n"
"    \"setsebool -P $ಬೂಲಿಯನ್=1\"\n"
"    "

#: ../src/httpd_unified.py:57
msgid ""
"If you want to allow httpd to execute cgi scripts and to unify HTTPD "
"handling of all content files."
msgstr ""

#: ../src/httpd_unified.py:58
msgid ""
"you must tell SELinux about this by enabling the 'httpd_unified' and "
"'http_enable_cgi' booleans"
msgstr ""
"'httpd_unified' ಹಾಗು 'http_enable_cgi' ಬೂಲಿಯನ್‌ಗಳನ್ನು ಶಕ್ತಗೊಳಿಸುವ ಮೂಲಕ ನೀವು "
"SELinux ಗೆ ಇದರ ಬಗ್ಗೆ ಹೇಳಬೇಕು"

#: ../src/httpd_write_content.py:30
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to $TARGET_PATH.\n"
"\n"
"    httpd scripts are not allowed to write to content without explicit\n"
"    labeling of all files.  If $TARGET_PATH is writable content. it needs\n"
"    to be labeled httpd_sys_rw_content_t or if all you need is append you "
"can label it httpd_sys_ra_content_t.   Please refer to 'man httpd_selinux' "
"for more information on setting up httpd and selinux.\n"
"    "
msgstr ""

#: ../src/httpd_write_content.py:40 ../src/qemu_blk_image.py:45
#: ../src/samba_share.py:45 ../src/swapfile.py:45 ../src/xen_image.py:48
msgid "You need to change the label on '$FIX_TARGET_PATH'"
msgstr "'$FIX_TARGET_PATH' ನಲ್ಲಿ ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸುವ ಅಗತ್ಯವಿದೆ'"

#: ../src/kernel_modules.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"kernel configuration.\n"
"    "
msgstr ""
"\n"
"    ನಿಮ್ಮ ಗಣಕವು ಗಂಭೀರವಾಗಿ ರಾಜಿ ಮಾಡಿಕೊಂಡಿರಬಹುದು! ಒಂದು ಕರ್ನಲ್ ಸಂರಚನೆಯನ್ನು "
"ಬದಲಾಯಿಸಲು $SOURCE_PATH ಪ್ರಯತ್ನಿಸಿದೆ.\n"
"    "

#: ../src/kernel_modules.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the way the kernel runs or to\n"
"    actually insert code into the kernel. All applications that need this\n"
"    access should have already had policy written for them.  If a "
"compromised\n"
"    application tries to modify the kernel this AVC will be generated. This "
"is a\n"
"    serious issue. Your system may very well be compromised.\n"
"    "
msgstr ""

#: ../src/kernel_modules.py:44 ../src/selinuxpolicy.py:44
msgid ""
"If you do not think $SOURCE_BASE_PATH should try $ACCESS access on "
"$TARGET_BASE_PATH."
msgstr ""

#: ../src/kernel_modules.py:45 ../src/selinuxpolicy.py:45
msgid ""
"you may be under attack by a hacker, since confined applications should not "
"need this access."
msgstr ""
"ನೀವು ಒಬ್ಬ ಹ್ಯಾಕರಿನಿಂದ ದಾಳಿಗೆ ಒಳಗಾಗಿದ್ದೀರಿ ಏಕೆಂದರೆ ನಿರ್ಬಂಧಿತ ಅನ್ವಯಗಳಿಗೆ ಈ "
"ನಿಲುಕಣೆಯ ಅಗತ್ಯವಿರುವುದಿಲ್ಲ."

#: ../src/leaks.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH access to a leaked $TARGET_PATH file "
"descriptor.\n"
"    "
msgstr ""
"\n"
"    ಸೋರಿಹೋದ $TARGET_PATH ಕಡತ ವಿವರಣೆಗಾರವನ್ನು ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ $TARGET_PATH "
"ಅನ್ನು SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/leaks.py:31
msgid ""
"\n"
"    SELinux denied access requested by the $SOURCE command. It looks like "
"this is either a leaked descriptor or $SOURCE output was redirected to a "
"file it is not allowed to access.  Leaks usually can be ignored since "
"SELinux is just closing the leak and reporting the error.  The application "
"does not use the descriptor, so it will run properly.  If this is a "
"redirection, you will not get output in the $TARGET_PATH.  You should "
"generate a bugzilla on selinux-policy, and it will get routed to the "
"appropriate package.  You can safely ignore this avc.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಆದೇಶದಿಂದ ಮನವಿಸಲ್ಲಿಸಲಾದ ನಿಲುಕಣೆಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ. ಬಹುಷಃ "
"ಇದು ಒಂದು ಸೋರಿಹೋದ ವಿವರಣೆಗಾರವಾಗಿದೆ(ಡಿಸ್ಕ್ರಿಪ್ಟರ್) ಅಥವ $SOURCE ಔಟ್‌ಪುಟ್ ಅನ್ನು "
"ಅದಕ್ಕೆ ನಿಲುಕಿಸಿಕೊಳ್ಳಲು ಅನುಮತಿ ಇರದೆ ಇರುವ ಒಂದು ಕಡತಕ್ಕೆ ಕಳುಹಿಸಲಾಗಿದೆ.  "
"ಸೋರುವಿಕೆಯನ್ನು ಸಾಮಾನ್ಯವಾಗಿ ಆಲಕ್ಷಿಸಬಹುದಾಗಿದೆ ಏಕೆಂದರೆ SELinux ಸೋರಿಕೆಯನ್ನು "
"ಮುಚ್ಚಿ ದೋಷವನ್ನು ವರದಿ ಮಾಡುತ್ತದೆ.  ಅನ್ವಯವು ವಿವರಣೆಗಾರರನ್ನು ಬಳಸುವುದಿಲ್ಲ, "
"ಆದ್ದರಿಂದ ಇದು ಸಮರ್ಪಕವಾಗಿ ಚಾಲನೆಗೊಳ್ಳುತ್ತದೆ.  ಎಲ್ಲಿಯಾದರೂ ಇದು ಮರು "
"ನಿರ್ದೇಶಿತಗೊಂಡಿದ್ದಾಗಿದ್ದರೆ, $TARGET_PATH ನಲ್ಲಿ ನಿಮಗೆ ಔಟ್‌ಪುಟ್ ದೊರೆಯುವುದಿಲ್ಲ.  "
"selinux-policy ಗಾಗಿ ನೀವು ಒಂದು ಬಗ್‌ಝಿಲ್ಲಾವನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ, ಸೂಕ್ತವಾದ "
"ಪ್ಯಾಕೇಜಿಗೆ ಕಳುಹಿಸಬೇಕಾಗುತ್ತದೆ.  ನೀವು ಈ avc ಯನ್ನು ನಿಶ್ಚಿಂತೆಯಿಂದ ಆಲಕ್ಷಿಸಬಹುದು.\n"
"    "

#: ../src/leaks.py:35
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"    "
msgstr ""
"\n"
"   ಈ ನಿಲುಕಣೆಯನ್ನು ಅನುಮತಿಸಲು ನೀವು ಒಂದು ಸ್ಥಳೀಯ ಪಾಲಿಸಿಯನ್ನು ಉತ್ಪಾದಿಸಬಹುದು\n"
"    - <a href=\"http://fedoraredhat.com/docs/selinux-faq-fc5/"
"#id2961385\">FAQ</a> ಅನ್ನು ನೋಡಿ\n"
"    "

#: ../src/leaks.py:42
msgid ""
"If you want to ignore $SOURCE_BASE_PATH trying to $ACCESS access the "
"$TARGET_BASE_PATH $TARGET_CLASS, because you believe it should not need this "
"access."
msgstr ""

#: ../src/leaks.py:43
msgid ""
"You should report this as a bug.  \n"
"You can generate a local policy module to dontaudit this access."
msgstr ""
"ನೀವು ಇದನ್ನು ಒಂದು ದೋಷ ಎಂದು ವರದಿ ಮಾಡಬೇಕು.\n"
"ಈ ನಿಲುಕಣೆಯನ್ನು dontaudit ಮಾಡಲು ನೀವು ಒಂದು ಸ್ಥಳೀಯ ಪಾಲಿಸಿಯನ್ನು ಉತ್ಪಾದಿಸಬಹುದು."

#: ../src/leaks.py:44
msgid ""
"# ausearch -x $SOURCE_PATH --raw | audit2allow -D -M my-$SOURCE\n"
"# semodule -i my-$SOURCE.pp"
msgstr ""

#: ../src/mmap_zero.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH attempted to mmap "
"low kernel memory.\n"
"    "
msgstr ""
"\n"
"    ನಿಮ್ಮ ಗಣಕವು ಗಂಭೀರವಾಗಿ ರಾಜಿ ಮಾಡಿಕೊಂಡಿರಬಹುದು! $SOURCE_PATH ಕಡಿಮೆ ಕರ್ನಲ್ "
"ಮೆಮೊರಿಯನ್ನು mmap ಮಾಡಲು ಪ್ರಯತ್ನಿಸಿದೆ.\n"
"    "

#: ../src/mmap_zero.py:33
msgid ""
"\n"
"    SELinux has denied the $SOURCE the ability to mmap low area of the "
"kernel \n"
"    address space.  The ability to mmap a low area of the address space is \n"
"    configured by /proc/sys/kernel/mmap_min_addr.  Preventing such mappings \n"
"    helps protect against exploiting null deref bugs in the kernel. All \n"
"    applications that need this access should have already had policy "
"written \n"
"    for them.  If a compromised application tries to modify the kernel, this "
"AVC \n"
"    would be generated. This is a serious issue. Your system may very well "
"be \n"
"    compromised.\n"
"    "
msgstr ""

#: ../src/mmap_zero.py:46
msgid ""
"If you do not think $SOURCE_PATH should need to mmap low memory in the "
"kernel."
msgstr ""

#: ../src/mmap_zero.py:47
msgid "you may be under attack by a hacker, this is a very dangerous access."
msgstr ""
"ನೀವು ಒಬ್ಬ ಹ್ಯಾಕರಿಂದ ದಾಳಿಗೆ ಒಳಗಾಗಿರಬಹುದು, ಇದು ಒಂದು ಅಪಾಯಕಾರಿಯಾದಂತಹ "
"ನಿಲುಕಣೆಯಾಗಿದೆ."

#: ../src/mounton.py:29
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting on the file or directory\n"
"    \"$TARGET_PATH\" (type \"$TARGET_TYPE\").\n"
"    "
msgstr ""
"\n"
"    SELinux ಕಡತವ್ಯವಸ್ಥೆಯನ್ನು  \"$TARGET_PATH\" (\"$TARGET_TYPE\" ಬಗೆ)\n"
"    ಕಡತ ಅಥವ ಕಡತಕೋಶದ ಮೇಲೆ ಆರೋಹಿಸದಂತೆ $SOURCE ಅನ್ನು ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/mounton.py:34
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting a filesystem on the file\n"
"    or directory \"$TARGET_PATH\" of type \"$TARGET_TYPE\". By default\n"
"    SELinux limits the mounting of filesystems to only some files or\n"
"    directories (those with types that have the mountpoint attribute). The\n"
"    type \"$TARGET_TYPE\" does not have this attribute. You can change the\n"
"    label of the file or directory.\n"
"    "
msgstr ""

#: ../src/mounton.py:43
msgid ""
"\n"
"    Changing the file_context to mnt_t will allow mount to mount the file "
"system:\n"
"    \"chcon -t mnt_t '$TARGET_PATH'.\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"mnt_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    file_context ಅನ್ನು mnt_t ಗೆ ಬದಲಾಯಿಸುವುದರಿಂದ ಕಡತವ್ಯವಸ್ಥೆಯನ್ನು ಆರೋಹಿಸಲು "
"ಮೌಂಟ್‍ಗೆ ಅನುಮತಿ ಒದಗಿಸುತ್ತದೆ\n"
"    \"chcon -t mnt_t '$TARGET_PATH'.\"\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.   \"semanage fcontext -a -t "
"mnt_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/mounton.py:48
msgid "If you want to allow $SOURCE_BASE_PATH to mount on $TARGET_BASE_PATH."
msgstr ""

#: ../src/mounton.py:49
msgid "you must change the labeling on $TARGET_PATH."
msgstr "$TARGET_PATH ನಲ್ಲಿ ನೀವು ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ."

#: ../src/mozplugger.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  mozplugger and "
"spice-xpi run applications within mozilla-plugins that require access to the "
"desktop, that the mozilla_plugin lockdown will not allow, so either you need "
"to turn off the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""

#: ../src/mozplugger.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi' or turn off enforcement of SELinux over the Firefox "
"plugins. setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""

#: ../src/mozplugger.py:43
msgid "you must turn off SELinux controls on the Firefox plugins."
msgstr ""
"ಫೈರ್ಫಾಕ್ಸ್ ಪ್ಲಗ್‌ಇನ್‌ಗಳಲ್ಲಿ ನೀವು SELinux ನಿಯಂತ್ರಕಗಳನ್ನು ಆಫ್ ಮಾಡಬೇಕಾಗುತ್ತದೆ."

#: ../src/mozplugger_remove.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ನಿಂದ ಸಲ್ಲಿಸಲಾದ ಮನವಿಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ. \n"
"    ಈ ಅನುಮತಿಯು $SOURCE ಗೆ ಅಗತ್ಯವಿದೆ ಎಂದು ನಿರೀಕ್ಷಿಸಿರಲಿಲ್ಲ ಹಾಗು ಇದನ್ನು\n"
"    ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲ್ಪಡುತ್ತದೆ. ಅನ್ವಯದ ನಿಶ್ಚಿತ ಆವೃತ್ತಿ\n"
"    ಅಥವ ಸಂರಚನೆಗಾಗಿ ಈ ಹೆಚ್ಚುವರಿ ಅನುಮತಿಯ ಅಗತ್ಯವಿರುವ ಸಾಧ್ಯತೆ ಇದೆ.\n"
"    "

#: ../src/mozplugger_remove.py:39
msgid ""
"\n"
"Either remove the mozplluger package by executing 'yum remove mozplugger'\n"
"Or turn off enforcement of SELinux over the Firefox plugins.\n"
"setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""
"\n"
"'yum remove mozplugger' ಅನ್ನು ಬಳಸಿಕೊಂಡು mozplluger ಪ್ಯಾಕೇಜ್ ಅನ್ನು ತೆಗೆದುಹಾಕಿ "
"ಅಥವ \n"
"ಫೈರ್ಫಾರ್ಕ್ಸ್ ಪ್ಲಗ್‌ಇನ್‌ಗಳ ಮೇಲೆ SELinux ಹೇರುವಿಕೆಯನ್ನು ಆಫ್ ಮಾಡಬೇಕು.\n"
"setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "

#: ../src/mozplugger_remove.py:44
msgid ""
"If you want to to continue using SELinux Firefox plugin containment rather "
"then using mozplugger package"
msgstr ""

#: ../src/mozplugger_remove.py:45
msgid "you must remove the mozplugger package."
msgstr "ನೀವು mozplugger ಪ್ಯಾಕೇಜನ್ನು ತೆಗೆದುಹಾಕಬೇಕಾಗುತ್ತದೆ."

#: ../src/openvpn.py:35
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. openvpn is allowed to read content in home directory if "
"it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/openvpn.py:41
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  \n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/openvpn.py:49
msgid ""
"If you want to mv $TARGET_BASE_PATH to standard location so that "
"$SOURCE_BASE_PATH can have $ACCESS access"
msgstr ""

#: ../src/openvpn.py:51
msgid ""
"If you want to modify the label on $TARGET_BASE_PATH so that "
"$SOURCE_BASE_PATH can have $ACCESS access on it"
msgstr ""

#: ../src/openvpn.py:55
msgid "you must move the cert file to the ~/.cert directory"
msgstr "ನೀವು cert ಕಡತವನ್ನು ~/.cert ಕೋಶಕ್ಕೆ ಜರುಗಿಸಬೇಕು"

#: ../src/openvpn.py:57 ../src/sshd_root.py:48
msgid "you must fix the labels."
msgstr "ನೀವು ಲೇಬಲ್‌ಗಳನ್ನು ಸರಿಪಡಿಸಬೇಕು."

#: ../src/public_content.py:31
msgid ""
"\n"
"    SELinux denied access to $TARGET_PATH requested by $SOURCE.\n"
"    $TARGET_PATH has a context used for sharing by a different program. If "
"you\n"
"    would like to share $TARGET_PATH from $SOURCE also, you need to\n"
"    change its file context to public_content_t.  If you did not intend to\n"
"    allow this access, this could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ನಿಂದ ಮನವಿಸಲ್ಲಿಸಲಾದ $TARGET_PATH ಗಾಗಿನ ನಿಲುಕಣೆಯನ್ನು SELinux "
"ನಿರಾಕರಿಸಿದೆ.\n"
"    $TARGET_PATH ವಿವಿಧ ಪ್ರೋಗ್ರಾಂ ನಡುವೆ ಹಂಚಿಕೊಳ್ಳಲು ಬಳಸಲಾಗುವ ಒಂದು "
"ಸನ್ನಿವೇಶವನ್ನು ಹೊಂದಿದೆ. If \n"
"ನೀವು $SOURCE ನಿಂದ $TARGET_PATH ಅನ್ನೂ ಸಹ ಹಂಚಿಕೆ ಮಾಡಲು ಬಯಸಿದಲ್ಲಿ, ಅದರ ಕಡತ "
"ಸನ್ನಿವೇಶವನ್ನು \n"
"    public_content_t ನಲ್ಲಿ ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  ನೀವು ಆ ಅನುಮತಿಗೆ ಬಯಸದೆ "
"ಇದ್ದಲ್ಲಿ\n"
"    ಇದು ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲಾಗುತ್ತದೆ.\n"
"    "

#: ../src/public_content.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -t public_content_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"public_content_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    chcon -t public_content_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"public_content_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/public_content.py:50
msgid "If you want to treat $TARGET_BASE_PATH as public content"
msgstr ""

#: ../src/public_content.py:51
msgid ""
"You need to change the label on $TARGET_BASE_PATH to public_content_t or "
"public_content_rw_t."
msgstr ""
"ನೀವು $TARGET_BASE_PATH ನಲ್ಲಿನ ಲೇಬಲ್ ಅನ್ನು  public_content_t or "
"public_content_rw_t ಗೆ ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ."

#: ../src/qemu_blk_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to the block device $TARGET_PATH.\n"
"    If this is a virtualization image, it needs to be labeled with a "
"virtualization file context (virt_image_t). You can relabel $TARGET_PATH to "
"be virt_image_t using chcon.  You also need to execute semanage fcontext -a -"
"t virt_image_t '$FIX_TARGET_PATH' to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    ಬ್ಲಾಕ್ ಸಾಧನ $TARGET_PATH ಗೆ qemu ಅನುಮತಿಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ.\n"
"    ಇದು ಒಂದು ವರ್ಚುವಲೈಸೇಶನ್ ಚಿತ್ರಿಕೆಯಾಗಿದ್ದರೆ,  ಒಂದು ವರ್ಚುವಲೈಸೇಶನ್ನ ಕಡತ "
"ಸನ್ನಿವೇಶ ಲೇಬಲ್ ಅನ್ನು ಹೊಂದಿರಬೇಕಾಗುತ್ತದೆ(virt_image_t).\n"
" ನೀವು chcon ಅನ್ನು ಬಳಸಿಕೊಂಡು $TARGET_PATH ಅನ್ನು virt_image_t ಗೆ ಮರು ಲೇಬಲ್ "
"ಮಾಡಬಹುದು.  ಗಣಕ ಪೂರ್ವನಿಯೋಜಿತಗಳಿಗೆ ಈ ಹೊಸ ಮಾರ್ಗವನ್ನು ಸೇರಿಸಲು semanage fcontext -"
"a -t virt_image_t '$FIX_TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಬೇಕಾಗುತ್ತದೆ\n"
"ನೀವು  $TARGET_PATH ಅನ್ನು ಒಂದು qemu ಚಿತ್ರಿಕೆಯಾಗಿ ಬಳಸಲು ಬಯಸಿಲ್ಲದೆ ಹೋಗಿದ್ದರೆ\n"
"    ಇದು ಒಂದು ದೋಷ ಅಥವ ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲಾಗುವುದು.\n"
"    "

#: ../src/qemu_blk_image.py:38 ../src/qemu_file_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t virt_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"virt_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    chcon -t virt_image_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು.\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"virt_image_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/qemu_blk_image.py:46
msgid ""
"# semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"

#: ../src/qemu_file_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to $TARGET_PATH.\n"
"    If this is a virtualization image, it has to have a file context label "
"of\n"
"    virt_image_t. The system is setup to label image files in directory./var/"
"lib/libvirt/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/"
"libvirt/images.\n"
"    If you really want to have your qemu image files in the current "
"directory, you can relabel $TARGET_PATH to be virt_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    $TARGET_PATH ಗೆ  qemu ಅನುಮತಿಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ.\n"
"    ಇದು ಒಂದು ವರ್ಚುವಲೈಸೇಶನ್ ಚಿತ್ರಿಕೆಯಾಗಿದ್ದರೆ, virt_image_t ನ ಒಂದು ಕಡತ "
"ಸನ್ನಿವೇಶ ಲೇಬಲ್ ಅನ್ನು ಹೊಂದಿರಬೇಕು.\n"
"    ಗಣಕವು directoryಕಡತಕೋಶ./var/lib/libvirt/images ದಲ್ಲಿನ ಕಡತಗಳಿಗೆ ಸಮರ್ಪಕವಾಗಿ "
"ಲೇಬಲ್ ಮಾಡಲು ಸಜ್ಜುಗೊಳಿಸಲಾಗಿದೆ.\n"
"    /var/lib/libvirt/images ಗೆ ನಿಮ್ಮ ಕಡತಗಳನ್ನು ನಕಲು ಮಾಡಲು ಸೂಚಿಸುತ್ತೇವೆ.\n"
"    ನಿಮ್ಮ xen ಚಿತ್ರಿಕಾ ಕಡತಗಳನ್ನು ಈಗಿನ ಕಡತಕೋಶದಲ್ಲಿ ನಿಜವಾಗಲೂ ಇರಿಸಲು ಬಯಸಿದಲ್ಲಿ, "
"ನೀವು$TARGET_PATH ಅನ್ನು chcon ಬಳಸಿಕೊಂಡು virt_image_t ಆಗಿ ಮರು ಲೇಬಲ್ ಮಾಡಬಹುದು.  "
"ಗಣಕ ಪೂರ್ವನಿಯೋಜಿತಗಳಿಗೆ ಈ ಹೊಸ ಮಾರ್ಗವನ್ನು ಸೇರಿಸಲು semanage fcontext -a -t "
"virt_image_t '$FIX_TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಬೇಕಾಗುತ್ತದೆ\n"
"ನೀವು  $TARGET_PATH ಅನ್ನು ಒಂದು qemu ಚಿತ್ರಿಕೆಯಾಗಿ ಬಳಸಲು ಬಯಸಿಲ್ಲದೆ ಹೋಗಿದ್ದರೆ\n"
"    ಇದು ಒಂದು ದೋಷ ಅಥವ ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲಾಗುವುದು.\n"
"    "

#: ../src/qemu_file_image.py:48
msgid "If $TARGET_BASE_PATH is a virtualization target"
msgstr ""

#: ../src/restorecon.py:57
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$TARGET_PATH', if this file is a "
"directory,\n"
"    you can recursively restore using restorecon -R '$TARGET_PATH'.\n"
"    "
msgstr ""
"\n"
"    ಈ ಕಡತಕ್ಕೆ ಗಣಕದ ಪೂರ್ವನಿಯೋಜಿತ ಸನ್ನಿವೇಶವನ್ನು ಮರುಸ್ಥಾಪಿಸಲು restorecon "
"ಆದೇಶವನ್ನು ಬಳಸಬಹುದಾಗಿದೆ.\n"
"    restorecon '$TARGET_PATH', ಇದು ಒಂದು ಕಡತಕೋಶವಾಗಿದ್ದಲ್ಲಿ,\n"
"    restorecon -R '$TARGET_PATH' ಅನ್ನು ಬಳಸಿಕೊಂಡು ಪುನರಾವರ್ತಿತವಾಗಿ "
"ಮರುಸ್ಥಾಪಿಸಬಹುದಾಗಿದೆ.\n"
"    "

#: ../src/restorecon.py:64
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled.  $TARGET_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$TARGET_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    File contexts can be assigned to a file in the following ways.\n"
"    <ul>\n"
"        <li>Files created in a directory receive the file context of the "
"parent directory by default.\n"
"        <li>The SELinux policy might override the default label inherited "
"from the parent directory by\n"
"            specifying a process running in context A which creates a file "
"in a directory labeled B\n"
"            will instead create the file with label C. An example of this "
"would be the dhcp client running\n"
"            with the dhclient_t type and creating a file in the directory /"
"etc. This file would normally\n"
"            receive the etc_t type due to parental inheritance but instead "
"the file\n"
"            is labeled with the net_conf_t type because the SELinux policy "
"specifies this.\n"
"        <li>Users can change the file context on a file using tools such as "
"chcon, or restorecon.\n"
"    </ul>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon.py:90
#, python-format
msgid ""
"If you want to fix the label. \n"
"$TARGET_PATH default label should be %s."
msgstr ""

#: ../src/restorecon.py:96
msgid ""
"you can run restorecon. The access attempt may have been stopped due to "
"insufficient permissions to access a parent directory in which case try to "
"change the following command accordingly."
msgstr ""

#: ../src/restorecon.py:105 ../src/restorecon_source.py:73
msgid "Restore\n"
"Context"
msgstr "ಸನ್ನಿವೇಶವನ್ನು\n"
"ಮರಳಿ ಸ್ಥಾಪಿಸು"

#: ../src/restorecon_source.py:39
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$SOURCE_PATH'.\n"
"    "
msgstr ""
"\n"
"    ಈ ಕಡತಕ್ಕೆ ಗಣಕದ ಪೂರ್ವನಿಯೋಜಿತ ಸನ್ನಿವೇಶವನ್ನು ಮರುಸ್ಥಾಪಿಸಲು restorecon "
"ಆದೇಶವನ್ನು ನೀವುಬಳಸಬಹುದಾಗಿದೆ\n"
"    restorecon '$SOURCE_PATH'.\n"
"    "

#: ../src/restorecon_source.py:45
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $SOURCE_PATH may\n"
"    be mislabeled.  $SOURCE_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$SOURCE_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon_source.py:60
#, python-format
msgid ""
"If you want to fix the label. \n"
"$SOURCE_PATH default label should be %s."
msgstr ""

#: ../src/restorecon_source.py:65
msgid "you can run restorecon."
msgstr "restorecon ಅನ್ನು ಚಲಾಯಿಸಬಹುದು."

#: ../src/rsync_data.py:31
msgid ""
"\n"
"    SELinux denied RSYNC access to $TARGET_PATH.\n"
"    If this is an RSYNC repository, it has to have a file context label of\n"
"    rsync_data_t. If you did not intend to use $TARGET_PATH as an RSYNC "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/rsync_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t rsync_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"rsync_data_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"     chcon -R -t rsync_data_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"rsync_data_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/rsync_data.py:45
msgid "If $TARGET_BASE_PATH should be shared via the RSYNC daemon"
msgstr ""

#: ../src/rsync_data.py:46
msgid "You need to change the label on $TARGET_BASE_PATH"
msgstr "$TARGET_BASE_PATH ನಲ್ಲಿ ನೀವು ಲೇಬಲ್ ಅನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ"

#: ../src/samba_share.py:27
msgid ""
"\n"
"    SELinux is preventing Samba ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    Samba ($SOURCE_PATH) \"$ACCESS\" ವು $TARGET_PATH ಅನ್ನು "
"ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆSELinux ತಡೆದಿದೆ. \n"
"    "

#: ../src/samba_share.py:31
msgid ""
"\n"
"    SELinux denied samba access to $TARGET_PATH.\n"
"    If you want to share this directory with samba it has to have a file "
"context label of\n"
"    samba_share_t. If you did not intend to use $TARGET_PATH as a samba "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    Please refer to 'man samba_selinux' for more information on setting up "
"Samba and SELinux.\n"
"    "
msgstr ""

#: ../src/samba_share.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t samba_share_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"samba_share_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    chcon -R -t samba_share_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"samba_share_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/samba_share.py:53
#, python-format
msgid ""
"# semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH%s'\n"
"# restorecon %s -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH%s'\n"
"# restorecon %s -v '$FIX_TARGET_PATH'"

#: ../src/sandbox_connect.py:33
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER within a sandbox.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, you need to use "
"a different sandbox type like sandbox_web_t or sandbox_net_t.  \n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/sandbox_connect.py:41
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಅನ್ನು $PORT_NUMBER ಗೆ ಸಂಪರ್ಕಿತಗೊಳ್ಳಲು ನೀವು ಬಯಸಿದಲ್ಲಿ, ಇದನ್ನು "
"ಚಲಾಯಿಸಿ\n"
"\n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    "

#: ../src/sandbox_connect.py:49
msgid ""
"you need to modify the sandbox type. sandbox_web_t or sandbox_net_t.\n"
"For example:\n"
"sandbox -X -t sandbox_net_t $SOURCE_PATH\n"
"Please read 'sandbox' man page for more details.\n"
msgstr ""

#: ../src/selinuxpolicy.py:29
msgid "\n"
"    Your system may be seriously compromised!\n"
"    "
msgstr "\n"
"    ನಿಮ್ಮ ಗಣಕವು ಗಂಭೀರವಾಗಿ ರಾಜಿ ಮಾಡಿಕೊಂಡಿರಬಹುದು!\n"
"    "

#: ../src/selinuxpolicy.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the selinux policy configuration.\n"
"    All applications that need this access should have already had policy\n"
"    written for them.  If a compromised application tries to modify the "
"SELinux\n"
"    policy this AVC will be generated. This is a serious issue. Your system\n"
"    may very well be compromised.\n"
"    "
msgstr ""

#: ../src/setenforce.py:27
msgid "If you did not directly cause this AVC through testing."
msgstr ""

#: ../src/setenforce.py:28
msgid "if you think that you might have been hacked"
msgstr "ನಿಮ್ಮನ್ನು ಹ್ಯಾಕ್ ಮಾಡಲಾಗಿದೆ ಎಂದು ನಿಮಗೆ ಅನ್ನಿಸಿದರೆ"

#: ../src/setenforce.py:32
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"SELinux enforcement.\n"
"    "
msgstr ""
"\n"
"    ನಿಮ್ಮ ಗಣಕವು ಗಂಭೀರವಾಗಿ ರಾಜಿ ಮಾಡಿಕೊಂಡಿರಬಹುದು! SELinux ಅನ್ನು ಮಾರ್ಪಡಿಸಲು "
"$SOURCE_PATH ಪ್ರಯತ್ನಿಸಿದೆ.\n"
"    "

#: ../src/setenforce.py:36
msgid ""
"\n"
"    SELinux has prevented $SOURCE from writing to a file under /sys/fs/"
"selinux.\n"
"    Files under /sys/fs/selinux control the way SELinux is configured.\n"
"    All programs that need to write to files under /sys/fs/selinux should "
"have already had policy\n"
"    written for them. If a compromised application tries to turn off SELinux\n"
"    this AVC will be generated. This is a serious issue. Your system may "
"very\n"
"    well be compromised.\n"
"    "
msgstr ""

#: ../src/setenforce.py:48
msgid "If you believe $SOURCE_PATH tried to disable SELinux."
msgstr ""

#: ../src/setenforce.py:49
msgid ""
"you may be under attack by a hacker, since confined applications should "
"never need this access."
msgstr ""
"ನೀವು ಒಬ್ಬ ಹ್ಯಾಕರಿನಿಂದ ದಾಳಿಗೆ ಒಳಗಾಗಿದ್ದೀರಿ ಏಕೆಂದರೆ ನಿರ್ಬಂಧಿತ ಅನ್ವಯಗಳಿಗೆ "
"ಎಂದಿಗೂ ಈ ನಿಲುಕಣೆಯ ಅಗತ್ಯವಿರುವುದಿಲ್ಲ."

#: ../src/sshd_root.py:36
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. sshd is allowed to read content in /root/.ssh directory "
"if it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/sshd_root.py:42
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.\n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/sshd_root.py:55
msgid "Restore Context"
msgstr "ಸನ್ನಿವೇಶವನ್ನು ಮರಳಿ ಸ್ಥಾಪಿಸು"

#: ../src/swapfile.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    $SOURCE_PATH \"$ACCESS\" ಅನ್ನು $TARGET_PATH ಗೆ SELinux ತಡೆದಿದೆ.\n"
"    "

#: ../src/swapfile.py:31
msgid ""
"\n"
"    SELinux denied $SOURCE access to $TARGET_PATH.\n"
"    If this is a swapfile, it has to have a file context label of\n"
"    swapfile_t. If you did not intend to use\n"
"    $TARGET_PATH as a swapfile, this message could indicate either a bug or "
"an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/swapfile.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -t swapfile_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"swapfile_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    chcon -t swapfile_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"swapfile_t '$FIX_TARGET_PATH'\"\n"
"    "

#: ../src/sys_module.py:28
msgid ""
"If you do not believe your $SOURCE_PATH should be modifying the kernel, by "
"loading kernel modules"
msgstr ""

#: ../src/sys_module.py:29
msgid "You might have been hacked."
msgstr "ನಿಮ್ಮನ್ನು ಹ್ಯಾಕ್ ಮಾಡಲಾಗಿರಬಹುದು."

#: ../src/sys_module.py:33
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to load a "
"kernel module.\n"
"    "
msgstr ""
"\n"
"    ನಿಮ್ಮ ಗಣಕವು ಗಂಭೀರವಾಗಿ ರಾಜಿ ಮಾಡಿಕೊಂಡಿರಬಹುದು! $SOURCE_PATH ಒಂದು ಕರ್ನಲ್ "
"ಘಟಕವನ್ನು ಲೋಡ್ ಮಾಡಲು ಪ್ರಯತ್ನಿಸಿದೆ.\n"
"    "

#: ../src/sys_module.py:37
msgid ""
"\n"
"    SELinux has prevented $SOURCE from loading a kernel module.\n"
"    All confined programs that need to load kernel modules should have "
"already had policy\n"
"    written for them. If a compromised application\n"
"    tries to modify the kernel this AVC will be generated. This is a serious\n"
"    issue. Your system may very well be compromised.\n"
"    "
msgstr ""

#: ../src/sys_module.py:46
msgid ""
"If you do not believe that $SOURCE_PATH should be attempting to modify the "
"kernel by loading a kernel module."
msgstr ""

#: ../src/sys_module.py:47
msgid "A process might be attempting to hack into your system."
msgstr ""
"ಒಂದು ಪ್ರಕ್ರಿಯೆಯು ನಿಮ್ಮ ವ್ಯವಸ್ಥೆಯನ್ನು ಹ್ಯಾಕ್ ಮಾಡಲು ಪ್ರಯತ್ನಿಸುತ್ತಿರಬಹುದು."

#: ../src/sys_resource.py:28
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH the \"sys_resource\" capability.\n"
"    "
msgstr ""

#: ../src/sys_resource.py:32
msgid ""
"\n"
"    Confined domains should not require \"sys_resource\". This usually means "
"that     your system is running out some system resource like disk space, "
"memory, quota etc. Please clear up the disk and this\n"
"    AVC message should go away. If this AVC continues after you clear up the "
"disk space, please report this as a bug.\n"
"    "
msgstr ""

#: ../src/sys_resource.py:39
msgid ""
"If you do not want processes to require capabilities to use up all the "
"system resources on your system;"
msgstr ""

#: ../src/sys_resource.py:40
msgid ""
"you need to diagnose why your system is running out of system resources and "
"fix the problem.\n"
"\n"
"According to /usr/include/linux/capability.h, sys_resource is required to:\n"
"\n"
"/* Override resource limits. Set resource limits. */\n"
"/* Override quota limits. */\n"
"/* Override reserved space on ext2 filesystem */\n"
"/* Modify data journaling mode on ext3 filesystem (uses journaling\n"
"   resources) */\n"
"/* NOTE: ext2 honors fsuid when checking for resource overrides, so\n"
"   you can override using fsuid too */\n"
"/* Override size restrictions on IPC message queues */\n"
"/* Allow more than 64hz interrupts from the real-time clock */\n"
"/* Override max number of consoles on console allocation */\n"
"/* Override max number of keymaps */\n"
msgstr ""

#: ../src/vbetool.py:30
msgid ""
"\n"
"    SELinux has prevented vbetool from performing an unsafe memory operation."
"\n"
"    "
msgstr ""
"\n"
"     vbetool ಒಂದು ಅಸುರಕ್ಷಿತ ಮೆಮೊರಿ ಕಾರ್ಯವನ್ನು ನಿರ್ವಹಿಸುವುದನ್ನು SELinux "
"ತಡೆಯುತ್ತಿದೆ.\n"
"    "

#: ../src/vbetool.py:34
msgid ""
"\n"
"SELinux denied an operation requested by $SOURCE, a program used\n"
"to alter video hardware state.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as vbetool.  This tool is used to\n"
"reset video state when a machine resumes from a suspend.  If your machine\n"
"is not resuming properly your only choice is to allow this\n"
"operation and reduce your system security against such malware.\n"
"\n"
"    "
msgstr ""

#: ../src/vbetool.py:45 ../src/wine.py:56
msgid ""
"\n"
"If you decide to continue to run the program in question you will need\n"
"to allow this operation.  This can be done on the command line by\n"
"executing:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"
msgstr ""
"\n"
"ಪ್ರಶ್ನೆಯಲ್ಲಿರುವ ಅನ್ವಯವನ್ನು ನೀವು ಚಲಾಯಿಸಲು ಬಯಸಿದಲ್ಲಿ ಈ ಕಾರ್ಯಕ್ಕೆ ನೀವು\n"
"ಅನುಮತಿಯನ್ನು ನೀಡಬೇಕಾಗುತ್ತದೆ.ಇದನ್ನು ನೆರವೇರಿಸಲು ಆಜ್ಞಾ ಸಾಲಿನಲ್ಲಿ ಈ ಕೆಳಗಿನ\n"
"ಆದೇಶವನ್ನು ಚಲಾಯಿಸಿ:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"

#: ../src/vbetool.py:55
msgid ""
"If you want to ignore this AVC because it is dangerous and your machine "
"seems to be working correctly."
msgstr ""

#: ../src/vbetool.py:56
msgid ""
"you must tell SELinux about this by enabling the vbetool_mmap_zero_ignore "
"boolean."
msgstr ""
"vbetool_mmap_zero_ignore ಬೂಲಿಯನ್ ಅನ್ನು ಶಕ್ತಗೊಳಿಸುವ ಮೂಲಕ ನೀವು SELinux ಗೆ ಇದರ "
"ಬಗ್ಗೆ ಹೇಳಬೇಕು."

#: ../src/vbetool.py:62 ../src/wine.py:73
msgid "Turn off memory protection"
msgstr "ಮೆಮೊರಿ ಸಂರಕ್ಷಣೆಯನ್ನು ಆಫ್ ಮಾಡು"

#: ../src/wine.py:31
msgid ""
"\n"
"    SELinux has prevented wine from performing an unsafe memory operation.\n"
"    "
msgstr ""
"\n"
"    $SOURCE ಸಂಪರ್ಕವಾದ ಒಂದು ಡೆಮನ್ ಆಗಿ ಚಲಾಯಿತಗೊಳ್ಳುವುದನ್ನು SELinux ತಡೆಯುತ್ತಿದೆ."
"\n"
"    "

#: ../src/wine.py:35
msgid ""
"\n"
"SELinux denied an operation requested by wine-preloader, a program used\n"
"to run Windows applications under Linux.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as wine.  If you were\n"
"attempting to run a Windows program your only choices are to allow this\n"
"operation and reduce your system security against such malware or to\n"
"refrain from running Windows applications under Linux.  If you were not\n"
"attempting to run a Windows application this indicates you are likely\n"
"being attacked by some for of malware or program trying to exploit your\n"
"system for nefarious purposes.\n"
"\n"
"Please refer to\n"
"\n"
"http://wiki.winehq.org/PreloaderPageZeroProblem\n"
"\n"
"Which outlines the other problems wine encounters due to its unsafe use\n"
"of memory and solutions to those problems.\n"
"\n"
"    "
msgstr ""

#: ../src/wine.py:66
msgid ""
"If you want to ignore this AVC because it is dangerous and your wine "
"applications are working correctly."
msgstr ""

#: ../src/wine.py:67
msgid ""
"you must tell SELinux about this by enabling the wine_mmap_zero_ignore "
"boolean."
msgstr ""
"wine_mmap_zero_ignore ಬೂಲಿಯನ್ ಅನ್ನು ಶಕ್ತಗೊಳಿಸುವ ಮೂಲಕ ನೀವು SELinux ಗೆ ಇದರ "
"ಬಗ್ಗೆ ಹೇಳಬೇಕು."

#: ../src/xen_image.py:27
msgid ""
"\n"
"    SELinux is preventing xen ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    $TARGET_PATH ಅನ್ನು ನಿಲುಕಿಸಿಕೊಳ್ಳದಂತೆ xen ($SOURCE_PATH) \"$ACCESS\" "
"ಅನ್ನು SELinux ತಡೆದಿದೆ. \n"
"    "

#: ../src/xen_image.py:31
msgid ""
"\n"
"    SELinux denied xen access to $TARGET_PATH.\n"
"    If this is a XEN image, it has to have a file context label of\n"
"    xen_image_t. The system is setup to label image files in directory /var/"
"lib/xen/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/xen/"
"images.\n"
"    If you really want to have your xen image files in the current "
"directory, you can relabel $TARGET_PATH to be xen_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a xen\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    $TARGET_PATH ಗೆ  xen ಅನುಮತಿಯನ್ನು SELinux ನಿರಾಕರಿಸಿದೆ.\n"
"    ಇದು ಒಂದು XEN ಚಿತ್ರಿಕೆಯಾಗಿದ್ದರೆ, xen_image_t ನ ಒಂದು ಕಡತ ಸನ್ನಿವೇಶ ಲೇಬಲ್ "
"ಅನ್ನು ಹೊಂದಿರಬೇಕು.\n"
"    ಗಣಕವು ಕೋಶ /var/lib/xen/images ದಲ್ಲಿನ ಕಡತಗಳಿಗೆ ಸಮರ್ಪಕವಾಗಿ ಲೇಬಲ್ ಮಾಡಲು "
"ಸಜ್ಜುಗೊಳಿಸಲಾಗಿದೆ\n"
"    /var/lib/xen/images ಗೆ ನಿಮ್ಮ ಕಡತಗಳನ್ನು ನಕಲು ಮಾಡಲು ಸೂಚಿಸುತ್ತೇವೆ.\n"
"    ನಿಮ್ಮ xen ಚಿತ್ರಿಕಾ ಕಡತಗಳನ್ನು ಈಗಿನ ಕಡತಕೋಶದಲ್ಲಿ ನಿಜವಾಗಲೂ ಇರಿಸಲು ಬಯಸಿದಲ್ಲಿ, "
"ನೀವು$TARGET_PATH ಅನ್ನು chcon ಬಳಸಿಕೊಂಡು xen_image_t ಆಗಿ ಮರು ಲೇಬಲ್ ಮಾಡಬಹುದು.  "
"ಗಣಕ ಪೂರ್ವನಿಯೋಜಿತಗಳಿಗೆ ಈ ಹೊಸ ಮಾರ್ಗವನ್ನು ಸೇರಿಸಲು semanage fcontext -a -t "
"xen_image_t '$FIX_TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸಬೇಕಾಗುತ್ತದೆ\n"
"ನೀವು  $TARGET_PATH ಅನ್ನು ಒಂದು xen ಚಿತ್ರಿಕೆಯಾಗಿ ಬಳಸಲು ಬಯಸಿಲ್ಲದೆ ಹೋಗಿದ್ದರೆ\n"
"    ಇದು ಒಂದು ದೋಷ ಅಥವ ಒಂದು ಒಳನುಸುಳುವಿಕಾ ಪ್ರಯತ್ನ ಎಂದು ಸೂಚಿಸಲಾಗುವುದು.\n"
"    "

#: ../src/xen_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t xen_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"xen_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    chcon -t xen_image_t '$TARGET_PATH' ಅನ್ನು ಕಾರ್ಯಗತಗೊಳಿಸುವ ಮೂಲಕ ಕಡತದ "
"ಸನ್ನಿವೇಶವನ್ನು ಬದಲಾಯಿಸಬಹುದು\n"
"   ಒಂದು ಸಂಪೂರ್ಣ ಮರುಲೇಬಲ್ ನಂತರವೂ ಗಣಕಗಳ ಕಡತವನ್ನು ಉಳಿಸಿಕೊಳ್ಳಲು ಅವುಗಳ "
"ಪೂರ್ವನಿಯೋಜಿತ ಸಂರಚನೆಯನ್ನು ಬದಲಾಯಿಸಬೇಕಾಗುತ್ತದೆ.  \"semanage fcontext -a -t "
"xen_image_t '$FIX_TARGET_PATH'\"\n"
"    "
